<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Abhishek Kumar">
<meta name="author" content="Meenu Patil">
<meta name="author" content="Pardeep Kumar">
<meta name="author" content="Anand Narain Singh">

<title>Exploring Elevational Patterns of Plant Species Richness: Insights from Western Himalayas</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract">Abstract</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#methodology" id="toc-methodology" class="nav-link" data-scroll-target="#methodology">Methodology</a>
  <ul class="collapse">
  <li><a href="#study-sites" id="toc-study-sites" class="nav-link" data-scroll-target="#study-sites">Study sites</a></li>
  <li><a href="#sec-checklist" id="toc-sec-checklist" class="nav-link" data-scroll-target="#sec-checklist">Species check-list</a></li>
  <li><a href="#distribution-ranges" id="toc-distribution-ranges" class="nav-link" data-scroll-target="#distribution-ranges">Distribution ranges</a></li>
  <li><a href="#species-richness" id="toc-species-richness" class="nav-link" data-scroll-target="#species-richness">Species richness</a></li>
  <li><a href="#data-analysis" id="toc-data-analysis" class="nav-link" data-scroll-target="#data-analysis">Data analysis</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#species-richness-across-sites" id="toc-species-richness-across-sites" class="nav-link" data-scroll-target="#species-richness-across-sites">Species richness across sites</a></li>
  <li><a href="#elevational-patterns-of-species-richness" id="toc-elevational-patterns-of-species-richness" class="nav-link" data-scroll-target="#elevational-patterns-of-species-richness">Elevational patterns of species richness</a></li>
  <li><a href="#comparison-with-null-model" id="toc-comparison-with-null-model" class="nav-link" data-scroll-target="#comparison-with-null-model">Comparison with null model</a></li>
  <li><a href="#elevational-pattern-of-residual-species-richness" id="toc-elevational-pattern-of-residual-species-richness" class="nav-link" data-scroll-target="#elevational-pattern-of-residual-species-richness">Elevational pattern of residual species richness</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion">Discussion</a></li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion">Conclusion</a></li>
  <li><a href="#acknowledgements" id="toc-acknowledgements" class="nav-link" data-scroll-target="#acknowledgements">Acknowledgements</a></li>
  <li><a href="#author-contributions" id="toc-author-contributions" class="nav-link" data-scroll-target="#author-contributions">Author contributions</a></li>
  <li><a href="#conflict-of-interest" id="toc-conflict-of-interest" class="nav-link" data-scroll-target="#conflict-of-interest">Conflict of interest</a></li>
  <li><a href="#funding" id="toc-funding" class="nav-link" data-scroll-target="#funding">Funding</a></li>
  <li><a href="#supplementary-tables" id="toc-supplementary-tables" class="nav-link" data-scroll-target="#supplementary-tables">Supplementary tables</a></li>
  <li><a href="#supplementary-figures" id="toc-supplementary-figures" class="nav-link" data-scroll-target="#supplementary-figures">Supplementary figures</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Exploring Elevational Patterns of Plant Species Richness: Insights from Western Himalayas</h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://akumar.netlify.app/">Abhishek Kumar</a> <a href="https://orcid.org/0000-0003-2252-7623" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Panjab University, Chandigarh
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Meenu-Patil">Meenu Patil</a> <a href="https://orcid.org/0000-0002-7664-7877" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Panjab University, Chandigarh
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Pardeep-Kumar-22">Pardeep Kumar</a> <a href="https://orcid.org/0000-0001-6707-1485" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Panjab University, Chandigarh
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Anand-Singh-15">Anand Narain Singh</a> <a href="https://orcid.org/0000-0002-0148-8680" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Panjab University, Chandigarh
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
  
    
  </div>
  

</header>

<section id="abstract" class="level1">
<h1>Abstract</h1>
<p>Understanding the patterns and drivers of species richness along elevational gradients is crucial for biogeographical, conservation and ecological research. This study aims to investigate the plant species richness across three sites at different elevations, examine the elevational patterns of species richness, compare the observed species richness with predictions of the mid-domain effect (MDE) null model, and assess the elevational pattern of residual species richness. We prepared a comprehensive database of elevational distribution of plant species occurring at selected sites by combining information from field visit and published literature. We used rigorous statistical analyses and null model simulations to elucidate species richness patterns along elevational gradients. Our study revealed substantial differences in plant species richness across the three study sites. While higher elevation site displayed a consistent decline in species richness with increasing elevation, others at lower and intermediate elevations exhibited a complex non-linear pattern. The entire elevational gradient demonstrated a non-linear unimodal pattern, with peak richness occurring around intermediate elevations. Further, the observed patterns showed deviations of varying magnitude and these deviations exhibited consistent non-linear relationship with elevation. These observations indicate that factors beyond the range constraints play a significant role in shaping species richness patterns along elevational gradients. Understanding these factors can aid in predicting and managing the impacts of ongoing environmental changes on elevational biodiversity. In conclusion, this study provides a comprehensive assessment of elevational patterns of plant species richness. By unravelling these patterns and investigating their underlying ecological drivers, we enhance our understanding of biodiversity dynamics along elevational gradients. This knowledge is crucial for informing effective conservation strategies and promoting the preservation of plant diversity in a rapidly changing world.</p>
<p><strong>Keywords:</strong> elevational gradients, species richness, mid-domain effect, residual species richness, western Himalaya, Morni Hills, Chail WLS, Churdhar WLS</p>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>More than two million living species have been scientifically documented so far and many more yet to be discovered <span class="citation" data-cites="COL2023">(<a href="#ref-COL2023" role="doc-biblioref">Bánki et al., 2023</a>)</span>. This enormous biodiversity is unevenly distributed on the planet and mountains harbour unusually higher number of species than any other terrestrial region <span class="citation" data-cites="Rahbek2019a">(<a href="#ref-Rahbek2019a" role="doc-biblioref">Rahbek et al., 2019</a>)</span>. Even within mountain ranges, species number shows striking variation as one moves from valleys to mountaintops <span class="citation" data-cites="Guo2013 Rahbek1995">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>)</span>. The change in number and diversity of species along a gradient of elevation has been referred as the elevational patterns of species richness. This field of research explores how species richness varies as one moves up or down a mountain or hillside. The distribution of species along elevational gradients has long intrigued ecologists <span class="citation" data-cites="Lomolino2001">(<a href="#ref-Lomolino2001" role="doc-biblioref">Lomolino, 2001</a>)</span> and remains a topic of active research in the fields of biogeography, conservation and ecology <span class="citation" data-cites="Guo2013 Lomolino2001 Rahbek1995">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-Lomolino2001" role="doc-biblioref">Lomolino, 2001</a>; <a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>)</span>. Understanding these patterns is essential for unravelling the factors that shape biodiversity and ecological processes in different ecosystems.</p>
<p>For a long time, it was believed that elevational gradients are mere reflections of latitudinal gradients <span class="citation" data-cites="Humboldt1805 Stevens1992">(<a href="#ref-Humboldt1805" role="doc-biblioref">Humboldt &amp; Bonpland, 1805</a>; <a href="#ref-Stevens1992" role="doc-biblioref">Stevens, 1992</a>)</span> and the number of species (species richness) decreases monotonically towards mountain tops <span class="citation" data-cites="Lomolino2001 Rahbek1995">(<a href="#ref-Lomolino2001" role="doc-biblioref">Lomolino, 2001</a>; <a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>)</span>. However, this decreasing pattern was challenged during late 20th century and a unimodal (hump-shaped) pattern was suggested to be more common than the monotonic decline <span class="citation" data-cites="Rahbek1995 Rahbek2005">(<a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>, <a href="#ref-Rahbek2005" role="doc-biblioref">2005</a>)</span>. During past decades, numerous studies investigated the elevational patterns of species richness for different taxa and diverse mountain ranges <span class="citation" data-cites="Guo2013 McCain2010 Rahbek1995">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>; <a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>)</span>. These studies frequently documented a unimodal relationship between elevation and species richness for a range of taxa <span class="citation" data-cites="McCain2010">(<a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>)</span> and it is widespread among many plant taxa <span class="citation" data-cites="Guo2013 McCain2010 Rahbek2005">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>; <a href="#ref-Rahbek2005" role="doc-biblioref">Rahbek, 2005</a>)</span>. However, the relationship between elevation and species richness varied considerably, with some studies finding a decreasing pattern <span class="citation" data-cites="Bisht2022 Trigas2013">(<a href="#ref-Bisht2022" role="doc-biblioref">Bisht et al., 2022</a>; <a href="#ref-Trigas2013" role="doc-biblioref">Trigas, Panitsa, &amp; Tsiftsis, 2013</a>)</span>, a plateau <span class="citation" data-cites="Kessler2011">(<a href="#ref-Kessler2011" role="doc-biblioref">Kessler, Kluge, Hemp, &amp; Ohlemüller, 2011</a>)</span>, or even an increase <span class="citation" data-cites="Kessler2011 Vetaas2002">(<a href="#ref-Kessler2011" role="doc-biblioref">Kessler et al., 2011</a>; <a href="#ref-Vetaas2002" role="doc-biblioref">Vetaas &amp; Grytnes, 2002</a>)</span> in species richness. While many taxa showed consistent unimodal elevational patterns, some taxa like birds <span class="citation" data-cites="McCain2009">(<a href="#ref-McCain2009" role="doc-biblioref">McCain, 2009</a>)</span> and ferns <span class="citation" data-cites="Kessler2011">(<a href="#ref-Kessler2011" role="doc-biblioref">Kessler et al., 2011</a>)</span> showed highly variable patterns of elevational species richness. The highly variable relationship between elevation and species richness indicates that the elevational patterns are not fully documented and understood yet. Therefore, there is still a scope to explore the elevational patterns of species richness for different taxa and mountain ranges <span class="citation" data-cites="Guo2013 Lomolino2001">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-Lomolino2001" role="doc-biblioref">Lomolino, 2001</a>)</span>.</p>
<p>While multiple biological, ecological and historical processes can shape elevational patterns of species richness <span class="citation" data-cites="Gaston2000 McCain2010">(<a href="#ref-Gaston2000" role="doc-biblioref">Gaston, 2000</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>)</span>, simple geometric constraints can also produce a unimodal pattern of species richness <span class="citation" data-cites="Colwell2000 Colwell2004">(<a href="#ref-Colwell2000" role="doc-biblioref">Colwell &amp; Lees, 2000</a>; <a href="#ref-Colwell2004" role="doc-biblioref">Colwell, Rahbek, &amp; Gotelli, 2004</a>)</span>. If random species with varying distribution ranges are placed in bounded domains, more species will overlap near the centre of the domains than the edges of the domains. Thus, a unimodal pattern of species richness can be achieved by solely due to random placement of species ranges <span class="citation" data-cites="Colwell2000">(<a href="#ref-Colwell2000" role="doc-biblioref">Colwell &amp; Lees, 2000</a>)</span>. This observation of richness peaks at intermediate elevations simply due to geometric constraints has been referred as the mid-domain effect (MDE). The prediction of MDE has been used as a null model because it operates solely due to random processes and did not consider any abiotic or biotic gradient <span class="citation" data-cites="Colwell2004 Vetaas2002">(<a href="#ref-Colwell2004" role="doc-biblioref">Colwell et al., 2004</a>; <a href="#ref-Vetaas2002" role="doc-biblioref">Vetaas &amp; Grytnes, 2002</a>)</span>. Available literature indicates variable agreement between observed species richness and the MDE predictions <span class="citation" data-cites="Colwell2004 Dunn2007">(<a href="#ref-Colwell2004" role="doc-biblioref">Colwell et al., 2004</a>; <a href="#ref-Dunn2007" role="doc-biblioref">R. R. Dunn, McCain, &amp; Sanders, 2007</a>)</span>, suggesting the influence of factors other than range constants in shaping the elevational patterns of species richness.</p>
<p>Understanding the elevational patterns of species distributions is a pre-requisite for identification of high diversity areas and has important implications for conservation and sustainability of fragile mountain ecosystems like Himalayas. Additionally, studying elevational patterns can inform conservation strategies in the face of ongoing environmental changes, such as climate change, which are expected to have profound effects on mountain ecosystems <span class="citation" data-cites="Rahbek2019a Steinbauer2018">(<a href="#ref-Rahbek2019a" role="doc-biblioref">Rahbek et al., 2019</a>; <a href="#ref-Steinbauer2018" role="doc-biblioref">Steinbauer et al., 2018</a>)</span>. While many studies explored elevational patterns of plant species richness in central and eastern Himalayas <span class="citation" data-cites="Bhattarai2004 Grau2007 Grytnes2002 Qian2022 Rana2019ecy Vetaas2002">(<a href="#ref-Bhattarai2004" role="doc-biblioref">Bhattarai, Vetaas, &amp; Grytnes, 2004</a>; <a href="#ref-Grau2007" role="doc-biblioref">Grau, Grytnes, &amp; Birks, 2007</a>; <a href="#ref-Grytnes2002" role="doc-biblioref">Grytnes &amp; Vetaas, 2002</a>; <a href="#ref-Qian2022" role="doc-biblioref">Qian, Kessler, &amp; Vetaas, 2022</a>; <a href="#ref-Rana2019ecy" role="doc-biblioref">Rana, Gross, &amp; Price, 2019</a>; <a href="#ref-Vetaas2002" role="doc-biblioref">Vetaas &amp; Grytnes, 2002</a>)</span>, studies in Western Himalayas remained limited <span class="citation" data-cites="Bisht2022 Chawla2008 Oommen2005">(<a href="#ref-Bisht2022" role="doc-biblioref">Bisht et al., 2022</a>; <a href="#ref-Chawla2008" role="doc-biblioref">Chawla et al., 2008</a>; <a href="#ref-Oommen2005" role="doc-biblioref">Oommen &amp; Shanker, 2005</a>)</span>. Through our research, we aspire to contribute to the broader understanding of biogeography, macroecology and conservation in the Western Himalayas. Therefore, the present analysis focused three protected areas, representing a large elevational gradient of 300–3600 m above mean sea level in Western Himalayas. By combining information from literature survey and published data, we seek to contribute to the growing body of research on elevational patterns of plant species richness in Western Himalayas. Specifically, the present study aims to (1) compare the plant species richness among different sites, (2) investigate the patterns of plant species richness across different elevational gradients, (3) compare the observed species richness with predictions of mid-domain effect null model across different elevational gradients, and (4) explore the elevational pattern of residual species richness (observed - predicted species richness). Our findings will contribute to enhance our understanding of the elevational patterns of plant species richness. Further, this knowledge can be used to inform conservation and management efforts, and to help us predict how plant communities will respond to future climate change.</p>
</section>
<section id="methodology" class="level1">
<h1>Methodology</h1>
<section id="study-sites" class="level2">
<h2 class="anchored" data-anchor-id="study-sites">Study sites</h2>
<p>Western Himalayas represent the prominent mountain ranges spanning over the Jammu &amp; Kashmir, Himachal Pradesh, Uttarakhand and parts of Punjab and Haryana in India. The present study focused on a combination of protected areas (wildlife sanctuaries) and their adjoining forested landscapes within the Western Himalayas. Specifically, we selected the Morni Hills (including Khol Hi-Raitan Wildlife Sanctuary), Chail Wildlife Sanctuary, and Churdhar Wildlife Sanctuary of Western Himalayas based on their ecological significance and accessibility. The Morni Hills (300–1500 m) is located in the the Panchkula district of Haryana, Chail WLS (900–2100 m) is shared by Solan and Shimla district of Himachal Pradesh, and Churdhar WLS (1600–3600 m) is shared by Sirmaur and Shimla districts of Himachal Pradesh (<a href="#fig-sites">Figure&nbsp;1</a>). Overall, these three sites provided a wider elevational gradient ranging from 300 m in the lower foothills to over 3600 m at the Churdhar Peak. The diverse ecological zones and topographic variation of these sites offered us an ideal setting to investigate the elevational patterns of plant species richness in Western Himalayas.</p>
<div id="fig-sites" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/site_map.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: Geographic location and elevation profile of Morni Hills, Chail WLS and Churdhar WLS in Western Himalayas. The Khol Hi-Raitan WLS is located in the western part of the Morni Hills. The inset map shows that the study sites are located in the southern part of Himachal Pradesh (HP) and northern part of Haryana (HR) state.</figcaption>
</figure>
</div>
<p>The selected study areas exhibit substantial variations in climate conditions due to large variations in elevation and topography (<a href="#fig-site-climate">Figure&nbsp;2</a>). Generally, the climate can be divided into three seasons, i.e., summer, monsoon, and winters. The climate varies from hot semi-arid (BSh) at foothills to monsoonal warm-summer humid continental climate (Dwb) near the Chur Peak. The Morni Hills represent the hot semi-arid (BSh) to monsoonal dry-winter humid subtropical climates (Cwa), the Chail WLS represent the monsoonal humid subtropical climate (Cwa) to monsoonal dry-winter subtropical highland climate (Cwb), and the the Churdhar WLS represent the monsoonal dry-winter subtropical highland climate (Cwb) to monsoonal warm-summer humid continental climate (Dwb) near the Chur Peak <span class="citation" data-cites="Beck2018">(<a href="#ref-Beck2018" role="doc-biblioref">Beck et al., 2018</a>)</span>.</p>
<div id="fig-site-climate" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/site_climate.svg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2: <span class="citation" data-cites="Walter1967">Walter &amp; Lieth (<a href="#ref-Walter1967" role="doc-biblioref">1967</a>)</span> climate diagrams for Morni Hills (topleft), Chail WLS (topright), Churdhar WLS (bottomleft) and All Sites (bottomright) showing the variation in average monthly temperature (shown in red) and precipitation (shown in blue) for 1970 to 2000. The text in top-left corner of each plot represent the name of the site with its elevation range in parenthesis and the period for which climate data is represented. Similarly, the text in top-right corner of each plot shows the mean annual temperature (°C) and mean annual precipitation (mm). The x-axis denotes the months and likely frost months are filled with sky blue. The left y-axis represent the temperature in °C (with red text), whereas the right y-axis depicts the precipitation in mm (with blue text). The minimum average temperature of the coldest month and maximum average temperature of the hottest month are represented on the left y-axis with black text. Above 100 mm precipitation, the scale of right y-axis is increased from 2 mm/°C to 20 mm/°C and this change in scale is marked by the black horizontal line in each plot. The area filled with red dots represent the dry period where the precipitation curve falls below the temperature curve. Similarly, the area filled with blue vertical lines shows the humid period where the precipitation curve falls above the temperature curve. The area filled in solid blue indicates the wet period where the precipitation curve falls above the black horizontal line represent the precipitation above 100 mm. The climate data was accessed from <span class="citation" data-cites="Fick2017">Fick &amp; Hijmans (<a href="#ref-Fick2017" role="doc-biblioref">2017</a>)</span>.</figcaption>
</figure>
</div>
<p>The Morni Hills (300–1500 m) harbours the tropical mixed dry deciduous forests at lower elevations and Siwalik Chir Pine forests at higher elevations. The Chail WLS (900–2100 m) comprise of subtropical Pine forests at lower elevations, followed by Oak forests and moist Deodar forests at higher elevations with occasional presence of Blue Pines. The Churdhar WLS (1600–3600) encompasses mixed coniferous forests at lower elevations followed by Kharsu Oak forests and alpine pastures at higher elevations <span class="citation" data-cites="Champion1968">(<a href="#ref-Champion1968" role="doc-biblioref">Champion &amp; Seth, 1968</a>)</span>. Thus, these sites represent a diverse range of plant communities, ranging from temperate forests of oak and rhododendron to alpine meadows adorned with vibrant wild-flowers. These diverse forests are home to some Endangered (<em>Aconitum heterophyllum</em>, <em>Angelica glauca</em>, <em>Cypripedium himalaicum</em>, <em>Dactylorhiza hatagirea</em>, <em>Picrorhiza kurroa</em>, <em>Taxus wallichiana</em> and <em>Trillium govanianum</em>), Vulnerable (<em>Cypripedium cordigerum</em>, <em>Malaxis muscifera</em> and <em>Paris polyphylla</em>), and Near Threatened (<em>Abies spectabilis</em>) vascular plants according to the recent assessment <span class="citation" data-cites="IUCN2023">(<a href="#ref-IUCN2023" role="doc-biblioref">IUCN, 2023</a>)</span>. Similarly, numerous endemic wild animals like Himalayan musk deer and Himalayan brown bear thrive in the study areas. The Chail WLS and Churdhar WLS are included in the Important Bird and Biodiversity Areas of the BirdLife International <span class="citation" data-cites="Rahmani2016">(<a href="#ref-Rahmani2016" role="doc-biblioref">Rahmani, Islam, &amp; Kasambe, 2016</a>)</span> and provide home to some threatened birds including the Cheer Pheasant (<em>Catreus wallichii</em>), Himalayan Monal (<em>Lophophorus impejanus</em>), Indian Vulture (<em>Gyps indicus</em>), Koklass Pheasant (<em>Pucrasia macrolopha</em>), Red-headed Vulture (<em>Sarcogyps calvus</em>), and White-rumped Vulture (<em>Gyps bengalensis</em>).</p>
</section>
<section id="sec-checklist" class="level2">
<h2 class="anchored" data-anchor-id="sec-checklist">Species check-list</h2>
<p>A comprehensive species check-list was compiled for each site by combining the information gathered from field surveys and literature surveys. We conducted 2-4 field visits of each site during 2018–2022 and recorded the identifiable plant species encountered on the treks followed by us. Unknown plants were photographed and identified in the lab with the help of literature and herbarium (PAN) of the Panjab University, Chandigarh and <a href="https://iiim.res.in/herbarium/index.htm">Janaki Ammal Herbarium (RRLH)</a> of Indian Institute of Integrative Medicine, Jammu. On-line resources like <a href="https://efloraofindia.com/">eFlora of India</a> and <a href="http://www.flowersofindia.net/">Flowers of India</a> were also consulted for plant species identification <span class="citation" data-cites="eFI2023 FOI2023">(<a href="#ref-eFI2023" role="doc-biblioref">eFI, 2023</a>; <a href="#ref-FOI2023" role="doc-biblioref">FOI, 2023</a>)</span>. Further, we conducted a systematic literature survey to identify the previously reported plant species from the selected sites. <a href="https://scholar.google.co.in/">Google Scholar</a> was chosen for identification of relevant studies for two reasons. First, it can retrieve the most obscure sources including studies published in local publication outlets and technical reports. Second, it can search within the full-text of available articles. Considering these advantages, we searched <a href="https://scholar.google.co.in/">Google Scholar</a> using the “Morni”, “Chail” and “Churdhar” as keywords during September 2021 and the search was again updated during August 2022. We recorded all the plant species reported from the identified accessible studies for each selected site. To prepare a complete check-list of reported plant species for all the sites, we updated our check-list by adding plant species that were collected and reported by earlier studies for Morni Hills <span class="citation" data-cites="Balkrishna2018a Balkrishna2018b Dhiman2020 Dhiman2021 Singh2014">(<a href="#ref-Balkrishna2018b" role="doc-biblioref">Balkrishna, Joshi, Srivastava, &amp; Shukla, 2018</a>; <a href="#ref-Balkrishna2018a" role="doc-biblioref">Balkrishna, Srivastava, Shukla, Mishra, &amp; Joshi, 2018</a>; <a href="#ref-Dhiman2020" role="doc-biblioref">Dhiman, Saharan, &amp; Jakhar, 2020</a>, <a href="#ref-Dhiman2021" role="doc-biblioref">2021</a>; <a href="#ref-Singh2014" role="doc-biblioref">Singh &amp; Vashistha, 2014</a>)</span>, Chail Wildlife Sanctuary <span class="citation" data-cites="Bhardwaj2014 Bhardwaj2017 Kumar2013">(<a href="#ref-Bhardwaj2017" role="doc-biblioref">Bhardwaj, 2017</a>; <a href="#ref-Bhardwaj2014" role="doc-biblioref">Bhardwaj, Verma, Rana, Thakur, &amp; Verma, 2014</a>; <a href="#ref-Kumar2013" role="doc-biblioref">Kumar, 2013</a>)</span>, and Churdhar Wildlife Sanctuary <span class="citation" data-cites="Choudhary2007 Choudhary2012 Gupta1998 Radha2019 Subramani2014 Thakur2021a">(<a href="#ref-Choudhary2007" role="doc-biblioref">A. K. Choudhary, Punam, Sharma, &amp; Chandel, 2007</a>; <a href="#ref-Choudhary2012" role="doc-biblioref">R. K. Choudhary &amp; Lee, 2012</a>; <a href="#ref-Gupta1998" role="doc-biblioref">Gupta, 1998</a>; <a href="#ref-Radha2019" role="doc-biblioref">Radha et al., 2019</a>; <a href="#ref-Subramani2014" role="doc-biblioref">Subramani, Kapoor, &amp; Goraya, 2014</a>; <a href="#ref-Thakur2021a" role="doc-biblioref">Thakur, Bisht, Kumar, &amp; Kumar, 2021</a>)</span>.</p>
<p>The <em>World Checklist of Vascular Plants (WCVP)</em> was followed for standardisation of all the botanical names and their authorities <span class="citation" data-cites="Govaerts2021">(<a href="#ref-Govaerts2021" role="doc-biblioref">Govaerts, Lughadha, Black, Turner, &amp; Paton, 2021</a>)</span>. It is based on the <em>International Plant Names Index (<a href="http://www.ipni.org">IPNI</a>)</em> and managed by the taxonomic experts at the Royal Botanic Gardens, Kew. It is considered superior to the traditionally used <em>The Plant List</em> (<a href="http://www.theplantlist.org/">TPL v1.1</a>) because it is expert reviewed and most importantly follows the <em>International Code of Nomenclature</em> <span class="citation" data-cites="Turland2018">(<a href="#ref-Turland2018" role="doc-biblioref">Turland et al., 2018</a>)</span>. Further, the WCVP follows the Angiosperm Phylogeny Group IV (APG IV) system of botanical classification <span class="citation" data-cites="APG2016">(<a href="#ref-APG2016" role="doc-biblioref">Angiosperm Phylogeny Group et al., 2016</a>)</span>. The WCVP database is updated daily and serves as a taxonomic backbone for the <em>Plants of the World Online</em> <span class="citation" data-cites="POWO2022">(<a href="#ref-POWO2022" role="doc-biblioref">POWO, 2022</a>)</span>. We manually screened and standardised each record of plant species using the <em>Plants of World Online</em> (<a href="https://powo.science.kew.org/" class="uri">https://powo.science.kew.org/</a>) portal during 2021–2022 <span class="citation" data-cites="POWO2022">(<a href="#ref-POWO2022" role="doc-biblioref">POWO, 2022</a>)</span>. All records of taxonomic names were updated by assigning the recently accepted names including the spelling variants and botanical authorities <span class="citation" data-cites="POWO2022">(<a href="#ref-POWO2022" role="doc-biblioref">POWO, 2022</a>)</span>. We generally accepted the infra-specific taxa (variety and subspecies) as species for the present analysis to maintain harmony among the taxa. Further, we recorded the distribution of each plant from the <span class="citation" data-cites="POWO2022">POWO (<a href="#ref-POWO2022" role="doc-biblioref">2022</a>)</span> and excluded all those taxa whose distribution was found outside of India for Morni Hills and outside of Western Himalaya for Chail WLS and Churdhar WLS. We manually corrected known distribution of some species (<em>Bauhinia variegata</em> as Introduced to Himalayas but Native to India). Recently, we updated the plant names and their families by matching with a static copy of WCVP (version 10 dated 27 October 2022) using the package <code>rWCVP</code> version 1.2.4 <span class="citation" data-cites="R-rWCVP">(<a href="#ref-R-rWCVP" role="doc-biblioref">Brown et al., 2023</a>)</span>.</p>
</section>
<section id="distribution-ranges" class="level2">
<h2 class="anchored" data-anchor-id="distribution-ranges">Distribution ranges</h2>
<p>The elevational distribution of each catalogued species was recorded from the recently compiled <em>Database of Vascular Plants of Himalaya</em> published on GBIF <span class="citation" data-cites="Rana2017 Rana2019gbif">(<a href="#ref-Rana2017" role="doc-biblioref">Rana &amp; Rawat, 2017</a>, <a href="#ref-Rana2019gbif" role="doc-biblioref">2019</a>)</span>. This dataset consist of more than 10,500 plant species compiled from published floras for the Himalayan region <span class="citation" data-cites="Rana2017 Rana2019gbif">(<a href="#ref-Rana2017" role="doc-biblioref">Rana &amp; Rawat, 2017</a>, <a href="#ref-Rana2019gbif" role="doc-biblioref">2019</a>)</span>. It included over 3,300 plant species from Himachal Pradesh with elevational distribution for about 3,000 species from the published floras <span class="citation" data-cites="Chowdhery1984 Collett1902 Duthie1903">(<a href="#ref-Chowdhery1984" role="doc-biblioref">Chowdhery &amp; Wadhwa, 1984</a>; <a href="#ref-Collett1902" role="doc-biblioref">Collett, 1902</a>; <a href="#ref-Duthie1903" role="doc-biblioref">Duthie, 1903–1915</a>)</span>. We accessed this dataset on 6 August 2022 through <a href="https://www.gbif.org/">GBIF</a> using the package <code>rgbif</code> version 3.7.7 <span class="citation" data-cites="R-rgbif">(<a href="#ref-R-rgbif" role="doc-biblioref">Chamberlain et al., 2023</a>)</span>. The plant names provided the authors of this dataset were standardised by matching with a static copy of WCVP (version 10 dated 27 October 2022) using the package <code>rWCVP</code> version 1.2.4 <span class="citation" data-cites="R-rWCVP">(<a href="#ref-R-rWCVP" role="doc-biblioref">Brown et al., 2023</a>)</span>. We filtered the elevational distribution of plants in the state of Himachal Pradesh and then, joined these elevational distributions to our plant check-list. This procedure provided elevational distribution for more than 1,000 plants. Similarly, elevational data for about 10 species was extracted from another published study <span class="citation" data-cites="Rana2019ecs">(<a href="#ref-Rana2019ecs" role="doc-biblioref">Rana, Price, &amp; Qian, 2019</a>)</span>. We excluded the remaining species (n = 228) whose elevational data was either not available or uncertain to us. Next, we manually screened the elevational data for each species. In case of duplicates, we considered maximum value for upper limit and minimum value for lower limit <span class="citation" data-cites="Rana2019ecs">(<a href="#ref-Rana2019ecs" role="doc-biblioref">Rana, Price, et al., 2019</a>)</span>. Further, we preferred the data from Himachal Pradesh over the neighbouring states or the entire Himalayas because our sites broadly falls under this Indian state. However, some species had their elevational distribution extending beyond the elevational range (300–3,600 m) of our sites. Therefore, we adjusted the minimum lower limit of distribution to the minimum elevation of study site, minimum upper limit of distribution to the minimum elevation of study site plus 100 m and maximum upper limit of distribution to maximum elevation of study site rounded to nearest hundred. Thus, the elevational distribution limits represented the ‘soft boundaries’ for each site <span class="citation" data-cites="Colwell1994">(<a href="#ref-Colwell1994" role="doc-biblioref">Colwell &amp; Hurtt, 1994</a>)</span>.</p>
</section>
<section id="species-richness" class="level2">
<h2 class="anchored" data-anchor-id="species-richness">Species richness</h2>
<p>Firstly, we re-sampled the elevation to 30 arc-sec (~1 km) raster with bilinear interpolation using the <code>resample()</code> function from the <code>terra</code> package version 1.7.29 <span class="citation" data-cites="R-terra">(<a href="#ref-R-terra" role="doc-biblioref">Hijmans, 2023</a>)</span>. This re-sampling updated the elevational extents of Morni Hills (300–1300 m), Chail WLS (900–2100 m), Churdhar WLS (1600–3400 m) and All Sites (300–3400 m). To estimate species richness at different elevations, we divided each elevational gradient into 100-m elevational bands for each site. Such 100-m elevational bands have been previously used to study the elevational patterns of species richness in plants <span class="citation" data-cites="Li2022 Qian2022 Rana2019ecs">(<a href="#ref-Li2022" role="doc-biblioref">Li et al., 2022</a>; <a href="#ref-Qian2022" role="doc-biblioref">Qian et al., 2022</a>; <a href="#ref-Rana2019ecs" role="doc-biblioref">Rana, Price, et al., 2019</a>)</span>. Thus, the species richness was estimated in 10 bands for Morni Hills, 12 bands for Chail WLS, 18 bands for Churdhar WLS, and 31 bands for All Sites combined. Each elevational band is represent by the upper elevational limit in both text and figures. To estimate the species richness for each elevational band, we used the range interpolation approach assuming that each species can be found everywhere between its elevational range <span class="citation" data-cites="Grytnes2002">(<a href="#ref-Grytnes2002" role="doc-biblioref">Grytnes &amp; Vetaas, 2002</a>)</span>. The range interpolation method have been widely used to study the elevational patterns of plant species richness <span class="citation" data-cites="Hu2016 Manish2017 Rana2019ecs">(<a href="#ref-Hu2016" role="doc-biblioref">Hu et al., 2016</a>; <a href="#ref-Manish2017" role="doc-biblioref">Manish et al., 2017</a>; <a href="#ref-Rana2019ecs" role="doc-biblioref">Rana, Price, et al., 2019</a>)</span>. Although this approach can introduce bias in estimation of species richness <span class="citation" data-cites="Hu2016">(<a href="#ref-Hu2016" role="doc-biblioref">Hu et al., 2016</a>)</span>, it is commonly applied for compensating the sampling problems and overall methodological consistency. Therefore, we assigned each species to all elevational bands occurring wholly or partly within its known elevational range <span class="citation" data-cites="Grytnes2002 Qian2022 Rana2019ecs">(<a href="#ref-Grytnes2002" role="doc-biblioref">Grytnes &amp; Vetaas, 2002</a>; <a href="#ref-Qian2022" role="doc-biblioref">Qian et al., 2022</a>; <a href="#ref-Rana2019ecs" role="doc-biblioref">Rana, Price, et al., 2019</a>)</span>. Then, the species richness was estimated as the total number of species present in each 100-m elevational band. Since this species richness correspond to the whole elevational band, it represents the <span class="math inline">\(\gamma\)</span>-diversity for each elevational band <span class="citation" data-cites="Lomolino2001">(<a href="#ref-Lomolino2001" role="doc-biblioref">Lomolino, 2001</a>)</span>.</p>
</section>
<section id="data-analysis" class="level2">
<h2 class="anchored" data-anchor-id="data-analysis">Data analysis</h2>
<p>Firstly, we aimed to compare species richness per 100-m elevational band across selected sites at different elevations. Since the number of 100-m elevational bands varied in our selected sites, we suspected substantial deviations from the assumptions of parametric test. Therefore, we assessed the assumptions of normality and homogeneity of variance for species richness before comparing mean species richness among selected sites. The Levene’s test indicated that species richness had significantly different variances among the selected sites (<em>F<sub>3,67</sub></em> = 11.33, <em>p</em> &lt; 0.001). Since our data did not meet the assumption of homogeneity of variances, we used the Welch’s one way analysis of variance <span class="citation" data-cites="Welch1951">(<a href="#ref-Welch1951" role="doc-biblioref">Welch, 1951</a>)</span> to compare the mean species richness per 100-m elevational band across the sites. Next, we conducted the Games-Howell post hoc test <span class="citation" data-cites="Games1976">(<a href="#ref-Games1976" role="doc-biblioref">Games &amp; Howell, 1976</a>)</span> to assess the differences between all unique pairwise comparisons. The Games-Howell post hoc test has been recommended for pairwise comparison of samples with unequal sample size and variances <span class="citation" data-cites="Ruxton2008">(<a href="#ref-Ruxton2008" role="doc-biblioref">Ruxton &amp; Beauchamp, 2008</a>)</span>. We implemented these statistical tests in <code>R</code> statistical environment version 4.3.0 <span class="citation" data-cites="R-base">(<a href="#ref-R-base" role="doc-biblioref">R Core Team, 2023</a>)</span> using the <code>R</code> package <code>rstatix</code> version 0.7.2 <span class="citation" data-cites="R-rstatix">(<a href="#ref-R-rstatix" role="doc-biblioref">Kassambara, 2023</a>)</span>.</p>
<p>Secondly, we aimed to explore the elevational patterns of plant species richness in Western Himalayas. A generalised linear model (GLM) was used to examine the relationship between plant species richness as response variable and elevation as predictor variable. The GLM framework allows to accommodate data with non-normal error distributions, model non-linear relationships, incorporate multiple predictors, and provide statistical inference <span class="citation" data-cites="Hilbe2014">(<a href="#ref-Hilbe2014" role="doc-biblioref">Hilbe, 2014</a>)</span>. Our response variable, plant species richness, was defined as the count of unique plant species observed within predefined elevation bands. We chose to use a Poisson distribution within the GLM due to the discrete and non-negative nature of our response variable. Additionally, the Poisson distribution is commonly used when modelling count data, as it accounts for the inherent heterogeneity and overdispersion observed in such discrete data <span class="citation" data-cites="Hilbe2014">(<a href="#ref-Hilbe2014" role="doc-biblioref">Hilbe, 2014</a>)</span>. Therefore, we used the GLM framework with a Poisson distribution and logarithmic-link function, which ensures that the fitted values are always non-negative. The GLMs were implemented using the <code>glm()</code> function from the <code>stats</code> package in <code>R</code> programming environment version 4.3.0 <span class="citation" data-cites="R-base">(<a href="#ref-R-base" role="doc-biblioref">R Core Team, 2023</a>)</span>.</p>
<p>We initially assessed the relationship between elevation and plant species richness using a scatter plot and calculated the Pearson’s correlation coefficient to explore the strength and direction of the association (<a href="#fig-richness-linear">Figure&nbsp;8</a>). To account for unimodal or non-linear relationship between species richness and elevation <span class="citation" data-cites="Guo2013 Rahbek1995">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>)</span>, we included polynomial terms for elevation of up-to fifth degree as predictors (<a href="#eq-full-mod">Equation&nbsp;1</a>).</p>
<p><span id="eq-full-mod"><span class="math display">\[
S \sim Elev + Elev^2 + Elev^3 + Elev^4 + Elev^5
\tag{1}\]</span></span></p>
<p>We followed a forward model selection approach (<a href="#tbl-model-selection">Table&nbsp;5</a>) and evaluated total six models starting from an intercept only null model (zero degree) to a full model with fifth degree polynomial elevation as predictor variables (<a href="#eq-full-mod">Equation&nbsp;1</a>). We measured the lack of fit of the model by calculating the Deviance (D), which is the deviance of the fitted model from the perfectly saturated model (<a href="#tbl-model-selection">Table&nbsp;5</a>). The Deviance (D) of a model is defined as the twice of the maximum log-likelihood (<span class="math inline">\(\mathcal{L}\)</span>) of the model. Although there is no true <span class="math inline">\(R^2\)</span> statistic for GLMs, the pseudo-<span class="math inline">\(R^2\)</span> or the deviance-squared (<span class="math inline">\(D^2\)</span>) of the model can be estimated by <a href="#eq-glm-r2">Equation&nbsp;2</a>:</p>
<p><span id="eq-glm-r2"><span class="math display">\[
D^2 = \frac{D_{null} - D_{resid}}{D_{null}}; \quad D^2_{adj} = 1 - \frac{(1 - D^2) \times (n-1)}{n - k - 1}
\tag{2}\]</span></span></p>
<p>where the <span class="math inline">\(D_{null}\)</span> is the Deviance of null model (model with only an intercept) and the <span class="math inline">\(D_{resid}\)</span> is the Deviance of the model under study (saturated model). Thus, a smaller value of <span class="math inline">\(D_{resid}\)</span> will have higher explanatory power and therefore, better will be the model. However, smaller sample size can bias the deviance-squared, therefore we adjusted the deviance-squared by applying a correction for small sample size (<a href="#eq-glm-r2">Equation&nbsp;2</a>). Additionally, we assessed the dispersion parameter (<span class="math inline">\(\phi\)</span>) since heterogeneous ecological data often have higher variance than the mean value of response (overdispersion) violating the assumption of Poisson GLM (<a href="#tbl-model-selection">Table&nbsp;5</a>). If there is overdispersion, then the quantity <span class="math inline">\(D/\phi\)</span> will follow a <span class="math inline">\(\chi^2\)</span> distribution with <span class="math inline">\(n - k\)</span> degrees of freedom and the estimator for <span class="math inline">\(\phi\)</span> will be:</p>
<p><span id="eq-glm-overdispersion"><span class="math display">\[
\phi = \frac{D}{n - k}
\tag{3}\]</span></span></p>
<p>where, <span class="math inline">\(D\)</span> is the residual deviance, <span class="math inline">\(n\)</span> is the total number of observations (sample size) and <span class="math inline">\(k\)</span> is the number of unknown parameters (predictors) in the fitted model. If the estimated values of <span class="math inline">\(\phi\)</span> in <a href="#eq-glm-overdispersion">Equation&nbsp;3</a> are close to 1, we can assume that there is little or no overdispersion <span class="citation" data-cites="Hilbe2014">(<a href="#ref-Hilbe2014" role="doc-biblioref">Hilbe, 2014</a>)</span>.</p>
<p>We used the information theory based model selection criteria to identify the best model from candidate models (<a href="#tbl-model-selection">Table&nbsp;5</a>). Akaike’s Information Criterion (AIC) has been most commonly used for model comparison in ecology and evolution <span class="citation" data-cites="Johnson2004">(<a href="#ref-Johnson2004" role="doc-biblioref">Johnson &amp; Omland, 2004</a>)</span>. However, it can bias model comparison when sample size is small in comparison to the number of estimated parameters <span class="citation" data-cites="Burnham2002">(<a href="#ref-Burnham2002" role="doc-biblioref">Burnham &amp; Anderson, 2002</a>)</span>. Therefore, we corrected the AIC by applying a sample size correction suggested by <span class="citation" data-cites="Hurvich1989">Hurvich &amp; Tsai (<a href="#ref-Hurvich1989" role="doc-biblioref">1989</a>)</span>, which can be mathematically represented by the <a href="#eq-aicc">Equation&nbsp;4</a>:</p>
<p><span id="eq-aicc"><span class="math display">\[
AIC = -2 \mathcal{L} + 2k; \quad
AICc = AIC + \frac{2k(k+1)}{n-k-1}
\tag{4}\]</span></span></p>
<p>where, <span class="math inline">\(k\)</span> is the number of parameters to be estimated by model and <span class="math inline">\(n\)</span> is the total number of response observations in the model. The model with lowest AICc value was considered as best model and models with a difference of two AICc units were considered equally competitive <span class="citation" data-cites="Burnham2002">(<a href="#ref-Burnham2002" role="doc-biblioref">Burnham &amp; Anderson, 2002</a>)</span>. Since our model are nested, we statistically compared the larger (more predictors) models with smaller (less predictors) models using a Deviance-based Chi-squared test, also known as the likelihood ratio test (<a href="#tbl-model-selection">Table&nbsp;5</a>). A significant <em>p</em>-value (i.e., <em>p</em> &lt; 0.05) suggests significant improvement in model fit when additional predictors were included, i.e., larger model is better than the smaller model.</p>
<p>Once the models were selected, a thorough evaluation of their performance was conducted to ensure the validity of the findings. Model evaluation involved assessing the assumptions of the selected models and diagnosing potential issues. We used simulated residuals (n = 1000) to visually assess model fit by plotting residuals against the model predictions <span class="citation" data-cites="Dunn1996">(<a href="#ref-Dunn1996" role="doc-biblioref">P. K. Dunn &amp; Smyth, 1996</a>)</span>. Then, the model was validated by analysing the dispersion and distributional assumptions of the fitted model. The simulated residuals from the defined distribution were tested against the residuals of the fitted model. Specifically, the uniformity was tested using the Kolmogorov-Smirnov (KS) test, dispersion was tested using the simulation-based dispersion test and outliers were tested by generating a simulation-based expectation for the outliers using the bootstrapping. If the dispersion test indicated significant over-/under-dispersion, we assessed the variance of of observed raw residuals against the variance of simulated residuals. The deviations in the model residuals were visually assessed by plotting the simulation-based residuals against the model predictions. This residual analysis and model validation was implemented using the <code>simulateResiduals()</code> function from the <code>DHARMa</code> package version 0.4.6 <span class="citation" data-cites="R-DHARMa">(<a href="#ref-R-DHARMa" role="doc-biblioref">Hartig, 2022</a>)</span>. Additionally, we performed the Deviance-based Chi-squared goodness-of-fit test to assess the overall goodness-of-fit of the selected model. A non-significant <em>p</em>-value indicates that there is no significant difference between the predictions of the model and observed data, i.e., the data is adequately fitted to the model. After assessment of goodness-of-fit for the selected models, we used Wald’s test to assess the significance of estimated regression coefficients.</p>
<p>Our third aim was to compare the observed species richness with predictions of a null model. A <strong>null model</strong> predicts the observed patterns solely due to operation of random processes <span class="citation" data-cites="Gotelli2009">(<a href="#ref-Gotelli2009" role="doc-biblioref">Gotelli et al., 2009</a>)</span>. The geometric constraint hypothesis or the mid-domain effect is a commonly used null model to evaluate species richness patterns <span class="citation" data-cites="Colwell1994 Grytnes2002">(<a href="#ref-Colwell1994" role="doc-biblioref">Colwell &amp; Hurtt, 1994</a>; <a href="#ref-Grytnes2002" role="doc-biblioref">Grytnes &amp; Vetaas, 2002</a>)</span>. This hypothesis predicts that the species richness patterns emerge from random distribution of species ranges. The <em>elevational range</em> of a species is defined as the difference between the highest and lowest elevation of its geographical distribution and the <em>elevational midpoint</em> as the mean of these two limits <span class="citation" data-cites="Stevens1992">(<a href="#ref-Stevens1992" role="doc-biblioref">Stevens, 1992</a>)</span>. We randomly generated elevational ranges equal to number of species in each site. The elevational ranges were generated by defining the geometric constraints of boundaries i.e., the lower and upper elevational limits corresponding to each study site <span class="citation" data-cites="Colwell2000">(Box 2 in <a href="#ref-Colwell2000" role="doc-biblioref">Colwell &amp; Lees, 2000</a>)</span>. These elevational limits were re-arranged so that the lower value should represent the lower limit whereas the higher value should represent the upper limit of elevational range. Then, the species richness was calculated by assuming that the species is continuously present within its elevational range. Next, we calculated species richness in each 100-m elevational band starting from minimum elevation to maximum elevation. This process was repeated 10,000 times to estimate the minimum, maximum, mean species richness and associated standard deviation for each elevational band. The mean species richness (S<sub>null</sub>) predicted by this null model was compared with the observed species richness (S). We used linear regression analysis to compare the strength of association between observed species richness and predictions of null models. If the observed species richness are well agreed with predictions of null model, then the slope value should be close to one.</p>
<p>Finally, we estimated the residual species richness (S<sub>res</sub>), representing the difference between observed (S) and predicted species richness (S<sub>null</sub>) for each 100-m elevational band. The residual species richness (S<sub>res</sub>) was calculated by subtracting the predicted species richness (generated by the mid-domain effect null model) from the observed species richness for each 100-m elevational band. Positive residuals indicated higher species richness than expected, while negative residuals indicated lower species richness than expected. We initially assessed the relationship between elevation and residual species richness using a scatter plot and calculated the Pearson’s correlation coefficient to explore the strength and direction of the association (<a href="#fig-sres-linear">Figure&nbsp;12</a>). To account for non-linear relationship between residual species richness and elevation, we included quadratic and cubic elevation terms as predictors. Then, we used polynomial linear regression to analysed the elevational pattern of residual species richness (S<sub>res</sub>). The goodness-of-fit for the polynomial models was assessed using the adjusted coefficient of determination (R<sup>2</sup><sub>adj</sub>) with higher values indicating better model fit. The significance of estimated regression coefficients was determined using the t-test and p-values less than 0.05 were considered as significant. All analyses were implemented in <code>R</code> statistical environment version 4.3.0 <span class="citation" data-cites="R-base">(<a href="#ref-R-base" role="doc-biblioref">R Core Team, 2023</a>)</span> and the package <code>tidyverse</code> version 2.0.0 was used for general data wrangling and visualisation <span class="citation" data-cites="R-tidyverse">(<a href="#ref-R-tidyverse" role="doc-biblioref">Wickham et al., 2019</a>)</span>.</p>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<section id="species-richness-across-sites" class="level2">
<h2 class="anchored" data-anchor-id="species-richness-across-sites">Species richness across sites</h2>
<p>Our initial check-list included over 2500 records of plant species across the selected sites. The standardisation of botanical names revealed that over 1000 botanical names were synonyms and we left with about 1400 unique botanical names. Further, screening for distribution showed that only 1385 were found within the study sites. These 1385 species belonged to 748 genera and 145 families (<a href="#tbl-flora-summary">Table&nbsp;1</a>). According to the <span class="citation" data-cites="POWO2022">POWO (<a href="#ref-POWO2022" role="doc-biblioref">2022</a>)</span> distribution, 1243 species were found to be Native whereas 142 species were Introduced to the region. The distribution of three species differed between India and Western Himalayas. For example, <em>Bauhinia variegata</em> and <em>Impatiens balsamina</em> are Native to India but the distribution of former is Doubtful or Introduced whereas that of latter is Introduced in Western Himalayas. Similarly, <em>Lysimachia arvensis</em> is Native to Western Himalayas but it is Introduced in India.</p>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-flora-summary" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;1: Summary of recorded vascular plant taxa from Morni Hills, Chail WLS and Churdhar WLS. The table includes the counts of unique taxa observed in each site, as well as the total number of unique taxa across all three sites. The taxa are categorised based on their taxonomic levels (species, genus, and family) and nativity (Introduced and Native).</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Taxa</th>
<th style="text-align: right;">Morni</th>
<th style="text-align: right;">Chail</th>
<th style="text-align: right;">Churdhar</th>
<th style="text-align: right;">Total</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Species</td>
<td style="text-align: right;">696</td>
<td style="text-align: right;">438</td>
<td style="text-align: right;">616</td>
<td style="text-align: right;">1385</td>
</tr>
<tr class="even">
<td style="text-align: left;">Genus</td>
<td style="text-align: right;">471</td>
<td style="text-align: right;">322</td>
<td style="text-align: right;">346</td>
<td style="text-align: right;">748</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Family</td>
<td style="text-align: right;">109</td>
<td style="text-align: right;">106</td>
<td style="text-align: right;">99</td>
<td style="text-align: right;">145</td>
</tr>
<tr class="even">
<td style="text-align: left;">Introduced</td>
<td style="text-align: right;">120</td>
<td style="text-align: right;">45</td>
<td style="text-align: right;">16</td>
<td style="text-align: right;">142</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Native</td>
<td style="text-align: right;">576</td>
<td style="text-align: right;">393</td>
<td style="text-align: right;">600</td>
<td style="text-align: right;">1243</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<p>A total of 696 species belonging to 471 genera and 109 families were recorded from Morni Hills. Among these 696 species, 576 species were Native and 120 species were Introduced. In Chail WLS, a total of 438 species belonging to 322 genera and 109 families were recorded. Among these 438 species, 393 species were Native and 45 species were Introduced. The Churdhar WLS represented a total of 616 species belonging to 346 genera and 99 families. Out of total 616 species, this region had total 600 Native and 16 Introduced species. Among all the selected sites, Morni Hills recorded maximum number of species followed by Churdhar WLS and Chail WLS (<a href="#tbl-flora-summary">Table&nbsp;1</a>). Similarly, the number of Introduced species were maximum in Morni Hills whereas it was minimum in Churdhar WLS.</p>
<div id="fig-venn" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/species_venn.svg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;3: Venn diagram illustrating the percent distribution of (a) species (n = 1385), (b) genera (n = 748), and (c) families (n = 145) across the selected sites. Each circle represents a specific site and the overlapping areas represent shared taxa, while the non-overlapping regions indicate unique taxa for each site. The diagram provides insights into the shared and site-specific taxa composition across the study sites.</figcaption>
</figure>
</div>
<p>Out of total 1385 species, 50 species were recorded from all the sites. Morni Hills, Chail WLS and Churdhar WLS had 515, 153, and 402 unique species, respectively. The Chail WLS and Churdhar WLS shared highest number of plant species (n = 134), whereas minimum species (n = 30) were shared by Morni Hills and Churdhar WLS (<a href="#fig-venn">Figure&nbsp;3</a> a). Similarly, about 97 genera were common to All Sites and Morni Hills comprised of greatest number of unique plant genera (n = 263) followed by Churdhar WLS (n = 138), whereas Chail WLS had lowest number of unique genera (n = 53). The Chail WLS shared highest number of genera (n = 86) with Morni Hills and Churdhar WLS, whereas only 25 genera were shared by the Morni Hills and Churdhar WLS (<a href="#fig-venn">Figure&nbsp;3</a> b). In terms of families, taxa from 65 families were common to all the sites and Morni Hills represent highest number of unique families. Morni Hills, Chail WLS and Churdhar WLS represented taxa from 24, 7, and 10 families, respectively. Generally, there was high similarity in taxa for Chail WLS and Churdhar WLS, whereas Morni Hills and Churdhar WLS exhibited unique taxonomic composition of plants. Among the families of the recorded species (<a href="#tbl-dom-families">Table&nbsp;4</a>), Fabaceae was most dominant with 133 species across All Sites followed by Asteraceae (n = 109), Poaceae (n = 93), Lamiaceae (n = 58) and Rosaceae (n = 43). Family Fabaceae (n = 105) was dominant in Morni Hills whereas family Asteraceae was dominant in Chail WLS (n = 41) and Churdhar WLS (n = 47).</p>
<p>Further, we observed significant differences in plant species richness per 100-m elevational band among the study sites along the elevational gradients (<a href="#fig-sr-comp">Figure&nbsp;4</a>). The average species richness per 100-m elevational band was highest for Churdhar WLS (mean = 313.12, SE = 29.58) followed by Morni Hills (mean = 163.30, SE = 17.70) and it was lowest for Chail WLS (mean = 180.92, SE = 8.69). When the data from All Sites was combined the average species richness was rose to 362.84 ± 28.77 per 100-m elevational band. Post-hoc tests revealed that the species richness per elevational band was significantly higher in Churdhar WLS as compared to Morni Hills (mean difference = 149.82, <em>p</em> &lt; 0.001) and Chail WLS (mean difference = 132.19, <em>p</em> &lt; 0.001). However, the mean species richness was not significantly differed between Morni Hills and Chail WLS (mean difference = 17.62, <em>p</em> = 0.653). The estimated omega squared (<span class="math inline">\(\omega^2\)</span> = 0.31) indicated that approximately 31% of the total variation in average species richness is attributable to differences among the three study sites.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-sr-comp" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-sr-comp-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;4: Comparison of species richness per 100-m elevational band among selected study sites. The Welch’s one way analysis of variance (ANOVA) was used to test the differences among the selected sites. The Games-Howell <em>post hoc</em> test was used to test the significane of difference between selected sites and the sites with different lower case letters are significantly different from each other (<em>p</em> &lt; 0.05).</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="elevational-patterns-of-species-richness" class="level2">
<h2 class="anchored" data-anchor-id="elevational-patterns-of-species-richness">Elevational patterns of species richness</h2>
<p>We were not able to retrieve elevational data for all recorded species and therefore, limited further analysis to only those species whose distribution data was available. Thus, our analyses were based on total 1,159 species from All Sites, which included 568 from Morni Hills, 377 from Chail WLS and 561 from Churdhar WLS. Out of the six evaluated candidate models for each site (<a href="#tbl-model-selection">Table&nbsp;5</a>), the quadratic model was considered as best model for Chail WLS and Churdhar WLS to describe the elevational patterns. The cubic model was chosen for Morni Hills whereas the quartic (fourth degree polynomial) model was selected for combined All Sites. Although the quadratic and cubic models were equally competitive for Morni Hills, we selected cubic model because it indicated better goodness-of-fit in terms of dispersion and residual diagnostics. Similarly, the quadratic model was chosen over the cubic model for Churdhar WLS due to better model diagnostics for quadratic model than the cubic model (<a href="#tbl-model-selection">Table&nbsp;5</a>). These selected models exhibited a good fit to the data as indicated by goodness-of-fit measures (<a href="#tbl-bmod-gof">Table&nbsp;2</a>). The quadratic, cubic and quartic models explained over 90% of total deviance for the selected sites. Additionally, the inspection of model residuals also suggested an adequate model fit as indicated by the QQ-plots (<a href="#fig-qqplot">Figure&nbsp;9</a>) with associated distribution tests and the residual plots to assess the deviation from the distribution assumptions (<a href="#fig-residual-plot">Figure&nbsp;10</a>). Although a significant under-dispersion was observed for the All Sites data, the variance of model residuals was not too large than the variance of simulated-residuals (<a href="#fig-dispersion-all">Figure&nbsp;11</a>).</p>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-bmod-gof" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;2: Summary of selected models to explain elevational patterns of species richness for each site. The plant species richness (S) was used as response variable and elevation (Elev) as predictor variable. Each model represent a generalised linear model (GLM) fitted using a Poisson distribution with log-link function. The table presents the Deviance explained (D<sup>2</sup>), adjusted Deviance explained (D<sup>2</sup><sub>adj</sub>), dispersion parameter (<span class="math inline">\(\phi\)</span>), corrected Akaike’s Information Criterion (AICc) for each model. The goodness-of-fit for each model was evaluated using a Deviance-based Chi-squared goodness-of-fit test and the Deviance (D<sub>resid</sub>), degrees of freedom (df<sub>resid</sub>) and associated p-values (p) are also reported.</caption>
<colgroup>
<col style="width: 8%">
<col style="width: 38%">
<col style="width: 4%">
<col style="width: 9%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 8%">
<col style="width: 9%">
<col style="width: 5%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Site</th>
<th style="text-align: left;">Model</th>
<th style="text-align: right;">D<sup>2</sup></th>
<th style="text-align: right;">D<sup>2</sup><sub>adj</sub></th>
<th style="text-align: right;"><span class="math inline">\(\phi\)</span></th>
<th style="text-align: right;">AICc</th>
<th style="text-align: right;">D<sub>resid</sub></th>
<th style="text-align: right;">df<sub>resid</sub></th>
<th style="text-align: right;">p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">S ~ Elev + Elev<sup>2</sup> + Elev<sup>3</sup></td>
<td style="text-align: right;">0.97</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">1.07</td>
<td style="text-align: right;">91.00</td>
<td style="text-align: right;">6.39</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">0.381</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">S ~ Elev + Elev<sup>2</sup></td>
<td style="text-align: right;">0.91</td>
<td style="text-align: right;">0.88</td>
<td style="text-align: right;">0.57</td>
<td style="text-align: right;">98.39</td>
<td style="text-align: right;">5.11</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">0.824</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">S ~ Elev + Elev<sup>2</sup></td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">156.64</td>
<td style="text-align: right;">14.06</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">0.521</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">S ~ Elev + Elev<sup>2</sup> + Elev<sup>3</sup> + Elev<sup>4</sup></td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">0.36</td>
<td style="text-align: right;">257.54</td>
<td style="text-align: right;">9.42</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">0.999</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<p>The species richness varied from 60 to 226 in Morni Hills, 124 to 220 in Chail WLS, 106 to 520 in Churdhar WLS and 79 to 588 across All Sites. The elevational patterns of plant species richness also differed among the selected sites. The elevational pattern of species richness followed a non-linear unimodal pattern for Morni Hills, Chail WLS, and All Sites combined (<a href="#fig-esr">Figure&nbsp;5</a>). This pattern showed an initial increase in plant species richness with increasing elevation, reaching a peak and then declines gradually at higher elevations. The species richness showed a peak around 800–900 m in Morni Hills, 1300–1400 m in Chail WLS and 1300–1400 m for All Sites. However, the Churdhar WLS showed a decreasing elevational pattern of plant species richness and the richness peak was observed around 1700–1800 in Churdhar WLS (<a href="#fig-esr">Figure&nbsp;5</a> <strong>c</strong>).</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-esr" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-esr-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;5: Elevational patterns of estimated species richness (solid line) plotted against the predictions of null model (dashed line) for (a) Morni Hills, (b) Chail WLS, (c) Churdhar WLS, and (d) All Sites. The data points (filled circles) represent the estimated species richness for each 100-m elevational band. The smooth line for estimated richness was fitted using the polynomial Poisson generalised linear model (GLM) with log-link function. The shaded region around the fitted solid line represents the 95% confidence intervals. The null model predictions represent the mean (dashed line) with minimum and maximum (grey shaded region) predicted species richness from 10,000 random simulations.</figcaption>
</figure>
</div>
</div>
</div>
<p>The Wald’s test suggested that elevation is indeed significantly associated with species richness. All estimated coefficients were significantly different from zero, except the Intercept for Morni Hills (<a href="#tbl-esr-coef">Table&nbsp;3</a>). The linear and cubic elevation terms were positively associated, whereas the quadratic and quartic elevation terms were negatively associated with species richness. The coefficient for linear elevation (Elev) was positive and highly significant for all the selected sites. However, the magnitude was highest for Morni Hills and it was lowest for Churdhar WLS, suggesting that the magnitude of increase in species richness is expected to be higher in Morni Hills than the Churdhar WLS (<a href="#tbl-esr-coef">Table&nbsp;3</a>). Similarly, the coefficient for quadratic elevation (Elev<sup>2</sup>) was negative and highly significant for all the sites, suggesting a significant decrease in species richness at higher elevations. The Chail WLS and Churdhar WLS included only quadratic elevation terms, suggesting that the pattern does not exhibit significant non-linear patterns beyond the quadratic term. However, the Morni Hills included a significant cubic elevation (Elev<sup>3</sup>) and All Sites quartic elevation (Elev<sup>4</sup>), suggesting significant non-linear relationship between elevation and species richness.</p>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-esr-coef" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;3: Summary of estimated coefficients (mean ± SE) from polynomial regression analysis of elevational patterns of plant species richness. The generalised linear model (GLM) was fitted by specifying the Poisson distribution with log-link function. The species richness (S) was used as response variable and elevation (Elev) as predictor variable. The significance of estimated coefficients was determined with Wald’s test and the significance levels <code>***</code>, <code>**</code> and <code>*</code> correspond to <em>p</em>-value of &lt;0.001, &lt;0.01 and &lt;0.05, respectively. The elevation was transformed to kilometre units before modelling.</caption>
<colgroup>
<col style="width: 10%">
<col style="width: 17%">
<col style="width: 18%">
<col style="width: 18%">
<col style="width: 17%">
<col style="width: 18%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Site</th>
<th style="text-align: left;">Intercept</th>
<th style="text-align: left;">Elev</th>
<th style="text-align: left;">Elev<sup>2</sup></th>
<th style="text-align: left;">Elev<sup>3</sup></th>
<th style="text-align: left;">Elev<sup>4</sup></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">-0.15 ± 0.92</td>
<td style="text-align: left;">14.95 ± 3.46***</td>
<td style="text-align: left;">-13.1 ± 4.12**</td>
<td style="text-align: left;">3.65 ± 1.56*</td>
<td style="text-align: left;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">1.65 ± 0.51**</td>
<td style="text-align: left;">4.68 ± 0.67***</td>
<td style="text-align: left;">-1.47 ± 0.21***</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">5.32 ± 0.36***</td>
<td style="text-align: left;">1.19 ± 0.3***</td>
<td style="text-align: left;">-0.4 ± 0.06***</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">1.8 ± 0.23***</td>
<td style="text-align: left;">8.49 ± 0.61***</td>
<td style="text-align: left;">-5.69 ± 0.56***</td>
<td style="text-align: left;">1.65 ± 0.21***</td>
<td style="text-align: left;">-0.19 ± 0.03***</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</section>
<section id="comparison-with-null-model" class="level2">
<h2 class="anchored" data-anchor-id="comparison-with-null-model">Comparison with null model</h2>
<p>The comparison of observed species richness (S) with predicted species richness (S<sub>null</sub>) from mid-domain effect null model indicated substantial deviations (<a href="#fig-mde-reg">Figure&nbsp;6</a>). The results of simple linear regression indicated that the null species richness (S<sub>null</sub>) did not fully explain the observed species richness (<a href="#tbl-mde-reg-coef">Table&nbsp;6</a>). The null species richness is significantly and positively associated with observed species richness for Chail WLS (<em>F<sub>1,10</sub></em> = 58.96, <em>p</em> &lt; 0.001) and All Sites combined (<em>F<sub>1,29</sub></em> = 170.20, <em>p</em> &lt; 0.001). However, there was no significant relationship for Morni Hills (<em>F<sub>1,8</sub></em> = 5.00, <em>p</em> = 0.056) and Churdhar WLS (<em>F<sub>1,16</sub></em> = 0.03, <em>p</em> = 0.862). The predicted null richness explained the 30.79%, 84.04% and 84.94% variation in observed species richness (S) for Morni Hills, Chail WLS and All Sites combined, respectively. On the other hand, the predicted species richness failed to explain any variation in observed species richness for Churdhar WLS (<a href="#tbl-mde-reg-coef">Table&nbsp;6</a>).</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-mde-reg" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-mde-reg-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;6: Scatterplot showing the relationship between observed and predicted species richness for (a) Morni Hills, (b) Chail WLS, (c) Churdhar WLS and (d) All Sites. The coloured solid line represent the fitted regression line and the shaded region represent the 95% confidence intervals. The estimated regression coefficients are represented as regression equation with the associated adjusted coefficient of determeination (R<sup>2</sup><sub>adj</sub>). The observed species richness was used as response variable and the predicted species richness was used as a predictor variable. The grey dashed line indicates the 1:1 line.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="elevational-pattern-of-residual-species-richness" class="level2">
<h2 class="anchored" data-anchor-id="elevational-pattern-of-residual-species-richness">Elevational pattern of residual species richness</h2>
<p>The residual species richness, i.e., differences in the observed and predicted null richness, varied from 1 to 446 across all the selected sites. These differences varied from 8 to 150 for Morni Hills, 1 to 93 for Chail WLS, 3 to 446 for Churdhar WLS and 1 to 161 for All Sites combined. The elevational pattern of residual species richness followed a cubic relationship for All Sites and a quadratic relationship for Morni Hills, Chail WLS and Churdhar WLS (<a href="#tbl-sres-coef">Table&nbsp;7</a>). Generally, the residual species richness tended to decrease at lower elevations, whereas it tended to increase at higher elevations. The decrease was more prominent for Churdhar WLS, whereas the increase was more prominent for Morni Hills. Overall, this quadratic linear model explained about 94%, 89% and 99% variation in residual species richness for Morni Hills, Chail WLS and Churdhar WLS, respectively (<a href="#fig-sres">Figure&nbsp;7</a>). However, the residual species richness exhibited a strong cubic relationship with elevation for the pooled data from All Sites (<a href="#fig-sres">Figure&nbsp;7</a> <strong>d</strong>). This cubic relationship indicated a slight increase in residual species richness at lower elevations, then a strong decrease at intermediate elevations and an increase at higher elevations. This cubic pattern of residual species richness suggests that the observed species richness substantially deviates from the predicted null richness at intermediate to higher elevations.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-sres" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-sres-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;7: Elevational patterns of residual species richness (S<sub>res</sub>) for (a) Morni Hills, (b) Chail WLS, (c) Churdhar WLS, and (d) All Sites. The residual species richness (S<sub>res</sub>) is calculated as the difference between observed species richness (S) and null species richness (S<sub>null</sub>). The null species richness (S<sub>null</sub>) is defined as the mean of predicted species richness from 10,000 replications of the mid-domain effect as null model. The data points (filled circles) represent the residual species richness for each 100-m elevational band. The smooth line was fitted using the polynomial linear regression with residual species richness (S<sub>res</sub>) as response variable and elevation as predictor variable. The shaded region around the fitted line represents the 95% confidence intervals. The estimated regression equation and the adjusted coefficient of determination (R<sup>2</sup><sub>adj</sub>) for each site is also presented. The grey dashed line indicates the zero residual species richness, i.e., observed species richness is perfectly identical to predicted species richness.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<p>In this study, we aimed to explore the elevational patterns of plant species richness. Additionally, we sought to compare the observed pattern with null predictions and examine the elevational patterns of residual plant species (i.e., the difference between observed and predicted values from the null model) to assess the influence of random processes on species richness. Our results indicated substantial variation in the observed plant taxa across sites along the elevational gradient (Question 1). Plant species richness generally exhibited a unimodal relationship with elevation, but a decreasing pattern was also observed (Question 2). Further, the observed species richness considerably deviated from the predictions of null model across the sites (Question 3). The magnitude and direction of these deviations (residual species richness) varied along the elevation gradient. Furthermore, the residual species richness demonstrated non-linear relationships with elevation (Question 4). Overall, our findings provide valuable insights into the relationship between elevational gradients and plant species richness.</p>
<p>Our data indicated substantial differences in observed plant taxa across the three sites along the elevational gradient. We recorded greater number of total and unique plant taxa at lower (Morni Hills) and upper (Churdhar WLS) elevations as compared to intermediate elevations (Chail WLS). Contrary to our expectations, we recorded minimum number of total and unique plant taxa at intermediate elevations (Chail WLS). The lower taxa records at intermediate elevations suggest dominance of fewer plant taxa. Alternatively, this region may under-explored and more plant taxa can be recorded in future surveys. Further, we observed no difference in the average species richness per elevational band for lower and intermediate elevations, but it was significantly higher at upper elevational gradients. The upper elevational gradient (Churdhar WLS) not only showed higher mean, but also exhibited large variance for plant species richness per elevational bands. This higher mean and variance at high elevation site indicate striking variation in plant species richness along elevational gradient. Thus, our data suggest substantial variation in plant taxa across the elevational gradient in western Himalayas.</p>
<p>The primary aim of this study was to investigate the elevational patterns of plant species in western Himalayas, focusing on three distinct sites along an elevational gradient. The results obtained from this study revealed interesting patterns, with sites at lower (Morni Hills) and intermediate (Chail WLS) elevations displaying a unimodal distribution of plant species along the elevational gradient, while the third site (Churdhar WLS) at higher elevations exhibited a decreasing pattern. Furthermore, the entire elevational gradient (All Sites) also demonstrated a unimodal relationship between elevation and plant species richness. The unimodal pattern showed that the species richness initially increases to reach a maximum richness (mid-elevational peak) and then decreases along the elevational gradient <span class="citation" data-cites="Guo2013 McCain2010">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>)</span>. This observed elevational pattern is consistent with previous research conducted in Himalayas <span class="citation" data-cites="Khuroo2011 Manish2017 Oommen2005 Rana2019ecy">(<a href="#ref-Khuroo2011" role="doc-biblioref">Khuroo, Weber, Malik, Reshi, &amp; Dar, 2011</a>; <a href="#ref-Manish2017" role="doc-biblioref">Manish et al., 2017</a>; <a href="#ref-Oommen2005" role="doc-biblioref">Oommen &amp; Shanker, 2005</a>; <a href="#ref-Rana2019ecy" role="doc-biblioref">Rana, Gross, et al., 2019</a>)</span> and other mountain ecosystems <span class="citation" data-cites="Guo2013 McCain2010">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>)</span>. In case of plants, this pattern has been observed for bryophytes <span class="citation" data-cites="Grau2007">(<a href="#ref-Grau2007" role="doc-biblioref">Grau et al., 2007</a>)</span>, pteridophytes <span class="citation" data-cites="Bhattarai2004 Kessler2011">(<a href="#ref-Bhattarai2004" role="doc-biblioref">Bhattarai et al., 2004</a>; <a href="#ref-Kessler2011" role="doc-biblioref">Kessler et al., 2011</a>)</span>, angiosperms <span class="citation" data-cites="Bryant2008 Manish2021">(<a href="#ref-Bryant2008" role="doc-biblioref">Bryant et al., 2008</a>; <a href="#ref-Manish2021" role="doc-biblioref">Manish, 2021</a>)</span>, orchids <span class="citation" data-cites="Djordjevic2022">(<a href="#ref-Djordjevic2022" role="doc-biblioref">Djordjević, Tsiftsis, Kindlmann, &amp; Stevanović, 2022</a>)</span>, woody plants <span class="citation" data-cites="Khuroo2011 Oommen2005">(<a href="#ref-Khuroo2011" role="doc-biblioref">Khuroo et al., 2011</a>; <a href="#ref-Oommen2005" role="doc-biblioref">Oommen &amp; Shanker, 2005</a>)</span>, and vascular plants <span class="citation" data-cites="Acharyabk2011 Chawla2008 Thorne2022 Vetaas2002">(<a href="#ref-Acharyabk2011" role="doc-biblioref">Acharya, Chettri, &amp; Vijayan, 2011</a>; <a href="#ref-Chawla2008" role="doc-biblioref">Chawla et al., 2008</a>; <a href="#ref-Thorne2022" role="doc-biblioref">Thorne et al., 2022</a>; <a href="#ref-Vetaas2002" role="doc-biblioref">Vetaas &amp; Grytnes, 2002</a>)</span>. Since three out of four studied elevational gradients observed this pattern, our analysis also support that the unimodal relationship between elevation and species richness is widespread among plants <span class="citation" data-cites="Guo2013 McCain2010 Rahbek1995">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>; <a href="#ref-Rahbek1995" role="doc-biblioref">Rahbek, 1995</a>)</span>.</p>
<p>One notable finding of the present study was the decreasing pattern of plant species richness at higher elevational gradient (Churdhar WLS). This finding aligns with earlier studies conducted in the Himalayas <span class="citation" data-cites="Bisht2022">(<a href="#ref-Bisht2022" role="doc-biblioref">Bisht et al., 2022</a>)</span> and other mountain ecosystems <span class="citation" data-cites="Di_Musciano2021 Kessler2011 Trigas2013">(<a href="#ref-Di_Musciano2021" role="doc-biblioref">Di Musciano et al., 2021</a>; <a href="#ref-Kessler2011" role="doc-biblioref">Kessler et al., 2011</a>; <a href="#ref-Trigas2013" role="doc-biblioref">Trigas et al., 2013</a>)</span>. Such decreasing elevational pattern of species richness has been also observed for microbes <span class="citation" data-cites="Bryant2008">(<a href="#ref-Bryant2008" role="doc-biblioref">Bryant et al., 2008</a>)</span>, bryophytes <span class="citation" data-cites="Rodriguez-Quiel2022">(<a href="#ref-Rodriguez-Quiel2022" role="doc-biblioref">Rodrı́guez-Quiel, Kluge, Mendieta-Leiva, &amp; Bader, 2022</a>)</span>, ferns <span class="citation" data-cites="Kessler2011">(<a href="#ref-Kessler2011" role="doc-biblioref">Kessler et al., 2011</a>)</span>, trees <span class="citation" data-cites="Homeier2010">(<a href="#ref-Homeier2010" role="doc-biblioref">Homeier, Breckle, Günter, Rollenbeck, &amp; Leuschner, 2010</a>)</span>, and vascular plants <span class="citation" data-cites="Bisht2022 Peters2016 Trigas2013">(<a href="#ref-Bisht2022" role="doc-biblioref">Bisht et al., 2022</a>; <a href="#ref-Peters2016" role="doc-biblioref">Peters et al., 2016</a>; <a href="#ref-Trigas2013" role="doc-biblioref">Trigas et al., 2013</a>)</span>. These observations suggest that decreasing patterns of plant species richness are not uncommon, though the unimodal pattern is frequently observed <span class="citation" data-cites="Guo2013 McCain2010">(<a href="#ref-Guo2013" role="doc-biblioref">Guo et al., 2013</a>; <a href="#ref-McCain2010" role="doc-biblioref">McCain &amp; Grytnes, 2010</a>)</span>. Thus, plant species richness may not always exhibit a unimodal relationship with elevation and further studies along elevational gradients would be valuable for understanding elevational patterns of species richness.</p>
<p>Another aim of the present study was to compare the observed species richness patterns to null predictions using the mid-domain effect (MDE) null model <span class="citation" data-cites="Colwell2000">(<a href="#ref-Colwell2000" role="doc-biblioref">Colwell &amp; Lees, 2000</a>)</span>. The MDE null model assumes that species ranges are randomly distributed within the given domains, and predicts a unimodal pattern of species richness along an elevational gradient, with peak richness occurring at the midpoint of the gradient. This pattern arises due to the geometric constraints imposed by the boundaries of the elevational range <span class="citation" data-cites="Colwell2004">(<a href="#ref-Colwell2004" role="doc-biblioref">Colwell et al., 2004</a>)</span>. Thus, the MDE model assumes that species richness is solely determined by the spatial constraints of the geographic range, independent of environmental factors <span class="citation" data-cites="Colwell2000">(<a href="#ref-Colwell2000" role="doc-biblioref">Colwell &amp; Lees, 2000</a>)</span>. Our results showed significant to substantial deviations in observed species richness from the predictions of the MDE null model. Specifically, the species richness patterns at upper (Churdhar WLS) and lower (Morni Hills) elevational gradients significantly deviated from the predictions of MDE model. This significant deviation might be due to presence of small-ranged species because smaller ranges experience less geometric constrain as compared to larger ranges <span class="citation" data-cites="Dunn2007">(<a href="#ref-Dunn2007" role="doc-biblioref">R. R. Dunn et al., 2007</a>)</span>. Further, the observed species richness across the entire elevational gradient (All Sites) was closely matched with predictions of MDE model followed by the intermediate elevational gradient (Chail WLS). These close agreements with predictions of MDE model might be due to increased geometric constraints on species ranges. The presence of large-ranged species at intermediate elevations and the extended scale for entire elevational gradient might have contributed to the increased geometric constraints on species ranges. Despite variable agreement <span class="citation" data-cites="Colwell2004">(<a href="#ref-Colwell2004" role="doc-biblioref">Colwell et al., 2004</a>)</span>, we observed considerable deviations in observed species richness from predictions of null model. Thus, our findings indicate that other factors beyond the range constraints may be playing a crucial role in shaping elevational patterns of species richness.</p>
<p>In addition to comparing observed patterns with null predictions, we aimed to investigate the elevational pattern of residual species richness (the difference between observed species richness and predictions of null model). The residual species richness represent the magnitude and direction of deviations from the predictions of mid-domain effect (MDE) null model. We observed positive residual species richness for sites representing intermediate (Chail WLS) and upper (Churdhar WLS) elevational gradient. This positive deviation indicated higher observed species richness than the predictions under null model of range constraints. On the other hand, we observed a general negative residual species richness for the entire elevational gradient (All Sites) and the site representing lower elevation gradient (Morni Hills). This negative residual species richness suggested lower than expected species richness based on range constraints alone. These deviations of variable magnitude and directions indicate the presence of other factors driving species richness rather than the range constraints alone.</p>
<p>An important finding of present study is that residual species richness showed significant quadratic relationships with elevation for the three individual sites and a significant cubic relationship for the entire elevational gradient. The significant quadratic relationships observed at the three individual sites suggest a complex pattern of species richness at smaller elevational gradients. This quadratic relationship indicates that the magnitude of deviation initially decreases with elevation, reaches a minimum, and then increases at higher elevations. Such relationship between elevation and magnitude of deviations indicate that species richness patterns are shaped by joint effects of different ecological processes occurring along the gradient. Further, the significant cubic relationship observed for the entire elevational gradient suggests a more complex and non-linear response of species richness to elevation. This cubic relationship indicates that the magnitude of deviation initially increases with elevation, reaches a peak, declines, and then potentially increases again at higher elevations. Such a pattern suggests the presence of additional factors or processes that influence species richness dynamics along the entire gradient. These factors could include large-scale climatic variables, historical and evolutionary processes, or interactions with other ecological variables that are not accounted for by the null model <span class="citation" data-cites="Gaston2000">(<a href="#ref-Gaston2000" role="doc-biblioref">Gaston, 2000</a>)</span>.</p>
<p>The observed species richness showed significant deviations from the predictions of null model and the residual species richness also demonstrated significant quadratic or cubic relationship with elevation. Both these observations indicate that the observed species richness pattern can not be achieved solely due to random processes. Our study highlights the limitations of the null model in fully capturing the observed elevational patterns of species richness. While null models provide a useful baseline for comparison and understanding broad-scale patterns, their assumptions and simplifications may not fully represent the complexity of ecological processes and interactions that shape elevational gradients. The observed deviations from the null model emphasise the need for considering additional environmental variables, biotic interactions, and historical factors when studying elevational patterns of species richness.</p>
<p>It is important to note that the observed elevational patterns in this study are specific to the selected sites and the study area. The present study used the range interpolation method to study the elevational patterns. Although it is widely used for biogeographical and macroecological studies <span class="citation" data-cites="Hu2016 Grytnes2002 Rana2019ecy Vetaas2002">(<a href="#ref-Grytnes2002" role="doc-biblioref">Grytnes &amp; Vetaas, 2002</a>; <a href="#ref-Hu2016" role="doc-biblioref">Hu et al., 2016</a>; <a href="#ref-Rana2019ecy" role="doc-biblioref">Rana, Gross, et al., 2019</a>; <a href="#ref-Vetaas2002" role="doc-biblioref">Vetaas &amp; Grytnes, 2002</a>)</span>, it has been criticised for its assumptions of continuous distribution of species <span class="citation" data-cites="Colwell2004 Dunn2007">(<a href="#ref-Colwell2004" role="doc-biblioref">Colwell et al., 2004</a>; <a href="#ref-Dunn2007" role="doc-biblioref">R. R. Dunn et al., 2007</a>)</span>. Further, the present study was based on the data compiled from the published flora, which can not be considered free from bias. Specifically, incomplete flora or biased estimation of distribution ranges can influence the observed patterns of species richness. Despite these limitations, data from published floras have been used for exploring the ecological and biogeographical patterns <span class="citation" data-cites="Di_Musciano2021 Qian2022 Vetaas2002">(<a href="#ref-Di_Musciano2021" role="doc-biblioref">Di Musciano et al., 2021</a>; <a href="#ref-Qian2022" role="doc-biblioref">Qian et al., 2022</a>; <a href="#ref-Vetaas2002" role="doc-biblioref">Vetaas &amp; Grytnes, 2002</a>)</span>. Overall, our study highlights the importance of considering the specific characteristics and dynamics of each site when studying elevational patterns. Further, the present study underscores the importance of considering the entire elevational gradient to capture the full range of ecological dynamics and complexities involved in shaping elevational patterns of species richness. Future studies should consider expanding the study area and incorporating additional environmental variables to gain a more comprehensive understanding of the drivers and mechanisms underlying elevational patterns of plant species. Environmental factors, geographical context, and historical factors can significantly influence elevational patterns, and caution should be exercised when generalising these findings to other regions or ecosystems.</p>
</section>
<section id="conclusion" class="level1">
<h1>Conclusion</h1>
<p>In conclusion, our study contributes to the understanding of plant species richness along elevational gradients. We observed considerable differences in species richness across the studied sites and identified a complex variable non-linear relationship between elevation and species richness. Further, the deviation from the predictions of the null model highlights the importance of factors beyond range constraints (mid-domain effect) in shaping elevational patterns of species richness. The analysis of residual species richness further elucidated elevations with positive and negative deviations, shedding light on the underlying ecological processes driving species richness. Furthermore, the quadratic or cubic relationship between elevation and residual species richness suggested the non-random distribution of plants influenced by multiple ecological processes. These findings highlight the limitations of the null model in fully capturing the complexities of elevational patterns and emphasise the need for incorporating additional ecological variables and mechanisms in future studies. Future research should focus on unravelling the determinants and specific mechanisms that contribute to these observed patterns. Such studies will enhance our understanding of the ecological processes shaping biodiversity and assist in developing effective conservation strategies in the face of environmental changes. The findings of this analysis contribute to our understanding of biodiversity patterns along elevation gradients and have implications for biodiversity conservation and ecosystem management. Overall, this study contributes to our understanding of the elevational patterns of plant species richness. The observed differences across study sites, the complex elevational patterns, the deviations from the null model predictions, and the quadratic or cubic relationships with elevation highlight the dynamic nature of species richness and the importance of considering multiple factors when studying elevational patterns of biodiversity.</p>
</section>
<section id="acknowledgements" class="level1 unnumbered">
<h1 class="unnumbered">Acknowledgements</h1>
<p>Authors are grateful to the Principal Chief Conservator of Forests (PCCF) of the Haryana Forest department and Himachal Pradesh Forest department for kindly granting the permission to visit the selected protected areas. We are also thankful to the Chairperson, Department of Botany, Panjab University, Chandigarh, for providing all the necessary facilities required for the work. We would like to express our heartfelt gratitude to <em>Sabir Hussain</em>, <em>Alok Sharma</em>, <em>Kamal</em>, <em>Pravesh</em> for their invaluable assistance and support during the fieldwork phase of this research. Additionally, we extend our appreciation to the staff and authorities of Forest Department of Himachal Pradesh who facilitated the logistics and permits required for the fieldwork.</p>
</section>
<section id="author-contributions" class="level1 unnumbered">
<h1 class="unnumbered">Author contributions</h1>
</section>
<section id="conflict-of-interest" class="level1">
<h1>Conflict of interest</h1>
<p>The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p>
</section>
<section id="funding" class="level1">
<h1>Funding</h1>
<p>University Grants Commission, Government of India, New Delhi is acknowledged for financial support in the form of Junior Research Fellowships to <em>Abhishek Kumar</em> [507/ (OBC) (CSIR-UGC NET DEC. 2016)], <em>Meenu Patil</em> [(492/ (CSIR-UGC NET JUNE 2017)], and <em>Pardeep Kumar</em> [443/ (CSIR-UGC NET DEC. 2017)].</p>
</section>
<section id="supplementary-tables" class="level1">
<h1>Supplementary tables</h1>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-dom-families" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;4: Top five dominant families of recorded vascular plant taxa across the sites. The total number of species within the respective family are shown in parenthesis.</caption>
<colgroup>
<col style="width: 23%">
<col style="width: 23%">
<col style="width: 27%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Morni</th>
<th style="text-align: left;">Chail</th>
<th style="text-align: left;">Churdhar</th>
<th style="text-align: left;">Total</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Fabaceae (105)</td>
<td style="text-align: left;">Asteraceae (41)</td>
<td style="text-align: left;">Asteraceae (47)</td>
<td style="text-align: left;">Fabaceae (133)</td>
</tr>
<tr class="even">
<td style="text-align: left;">Asteraceae (57)</td>
<td style="text-align: left;">Fabaceae (34)</td>
<td style="text-align: left;">Poaceae (39)</td>
<td style="text-align: left;">Asteraceae (109)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Poaceae (46)</td>
<td style="text-align: left;">Poaceae (25)</td>
<td style="text-align: left;">Rosaceae (38)</td>
<td style="text-align: left;">Poaceae (93)</td>
</tr>
<tr class="even">
<td style="text-align: left;">Lamiaceae (32)</td>
<td style="text-align: left;">Lamiaceae (22)</td>
<td style="text-align: left;">Lamiaceae (30)</td>
<td style="text-align: left;">Lamiaceae (58)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Malvaceae (28)</td>
<td style="text-align: left;">Rosaceae (20)</td>
<td style="text-align: left;">Ranunculaceae (29)</td>
<td style="text-align: left;">Rosaceae (43)</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-model-selection" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;5: Summary of evaluated candidate models to explore the relationship between elevation and plant species richness. The candidate models included zero degree intercept only (M0), first degree linear (M1), second degree quadratic (M2), third degree cubic (M3), fourth degree quartic (M4) and fifth degree quintic (M5) polynomial models. Each model represent a generalised linear model (GLM) fitted using a Poisson distribution with log-link function. The table presents the Deviance explained (D<sup>2</sup>), adjusted Deviance explained (D<sup>2</sup><sub>adj</sub>), dispersion parameter (<span class="math inline">\(\phi\)</span>), corrected Akaike’s Information Criterion (AICc) and difference in AICc from top model (Δ AICc) for each model. Each subsequent model was compared using the Deviance-based Chi-square test (likelihood ratio test) and the Deviance (D<sub>resid</sub>), degrees of freedom (df<sub>resid</sub>) and associated p-values (p) are also presented. The models are arranged in increasing order of complexity (zero degree to fifth degree polynomial) for each site, allowing comparison and identification of the most parsimonious and well-fitting model.</caption>
<colgroup>
<col style="width: 10%">
<col style="width: 7%">
<col style="width: 5%">
<col style="width: 11%">
<col style="width: 8%">
<col style="width: 9%">
<col style="width: 15%">
<col style="width: 10%">
<col style="width: 11%">
<col style="width: 8%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Site</th>
<th style="text-align: left;">Model</th>
<th style="text-align: right;">D<sup>2</sup></th>
<th style="text-align: right;">D<sup>2</sup><sub>adj</sub></th>
<th style="text-align: right;"><span class="math inline">\(\phi\)</span></th>
<th style="text-align: right;">AICc</th>
<th style="text-align: right;">Δ AICc</th>
<th style="text-align: right;">D<sub>resid</sub></th>
<th style="text-align: right;">df<sub>resid</sub></th>
<th style="text-align: right;">p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">M0</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">-0.13</td>
<td style="text-align: right;">22.11</td>
<td style="text-align: right;">270.10</td>
<td style="text-align: right;">179.57</td>
<td style="text-align: right;">198.99</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">M1</td>
<td style="text-align: right;">0.48</td>
<td style="text-align: right;">0.33</td>
<td style="text-align: right;">12.99</td>
<td style="text-align: right;">178.22</td>
<td style="text-align: right;">87.69</td>
<td style="text-align: right;">103.90</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">M2</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">0.91</td>
<td style="text-align: right;">1.70</td>
<td style="text-align: right;">90.52</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">11.92</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">M3</td>
<td style="text-align: right;">0.97</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">1.07</td>
<td style="text-align: right;">91.00</td>
<td style="text-align: right;">0.47</td>
<td style="text-align: right;">6.39</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">0.019</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">M4</td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.96</td>
<td style="text-align: right;">0.73</td>
<td style="text-align: right;">97.26</td>
<td style="text-align: right;">6.73</td>
<td style="text-align: right;">3.65</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">0.098</td>
</tr>
<tr class="even">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">M5</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.38</td>
<td style="text-align: right;">110.12</td>
<td style="text-align: right;">19.59</td>
<td style="text-align: right;">1.51</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">0.144</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">M0</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">-0.10</td>
<td style="text-align: right;">5.13</td>
<td style="text-align: right;">143.14</td>
<td style="text-align: right;">44.75</td>
<td style="text-align: right;">56.46</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">M1</td>
<td style="text-align: right;">0.04</td>
<td style="text-align: right;">-0.17</td>
<td style="text-align: right;">5.41</td>
<td style="text-align: right;">143.71</td>
<td style="text-align: right;">45.31</td>
<td style="text-align: right;">54.09</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">0.124</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">M2</td>
<td style="text-align: right;">0.91</td>
<td style="text-align: right;">0.88</td>
<td style="text-align: right;">0.57</td>
<td style="text-align: right;">98.39</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">5.11</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">M3</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">0.90</td>
<td style="text-align: right;">0.43</td>
<td style="text-align: right;">101.43</td>
<td style="text-align: right;">3.03</td>
<td style="text-align: right;">3.43</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.195</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">M4</td>
<td style="text-align: right;">0.95</td>
<td style="text-align: right;">0.91</td>
<td style="text-align: right;">0.37</td>
<td style="text-align: right;">106.90</td>
<td style="text-align: right;">8.50</td>
<td style="text-align: right;">2.62</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">0.367</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">M5</td>
<td style="text-align: right;">0.96</td>
<td style="text-align: right;">0.90</td>
<td style="text-align: right;">0.41</td>
<td style="text-align: right;">115.53</td>
<td style="text-align: right;">17.13</td>
<td style="text-align: right;">2.44</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">0.677</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">M0</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">-0.06</td>
<td style="text-align: right;">53.68</td>
<td style="text-align: right;">1049.65</td>
<td style="text-align: right;">896.40</td>
<td style="text-align: right;">912.53</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">M1</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">0.93</td>
<td style="text-align: right;">3.68</td>
<td style="text-align: right;">198.49</td>
<td style="text-align: right;">45.25</td>
<td style="text-align: right;">58.82</td>
<td style="text-align: right;">16</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">M2</td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.94</td>
<td style="text-align: right;">156.64</td>
<td style="text-align: right;">3.40</td>
<td style="text-align: right;">14.06</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">M3</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.52</td>
<td style="text-align: right;">153.24</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">7.30</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">0.009</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">M4</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.50</td>
<td style="text-align: right;">156.37</td>
<td style="text-align: right;">3.12</td>
<td style="text-align: right;">6.50</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">0.372</td>
</tr>
<tr class="even">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">M5</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">0.53</td>
<td style="text-align: right;">160.92</td>
<td style="text-align: right;">7.67</td>
<td style="text-align: right;">6.41</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">0.767</td>
</tr>
<tr class="odd">
<td style="text-align: left;">All</td>
<td style="text-align: left;">M0</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">-0.03</td>
<td style="text-align: right;">78.84</td>
<td style="text-align: right;">2603.04</td>
<td style="text-align: right;">2345.50</td>
<td style="text-align: right;">2365.19</td>
<td style="text-align: right;">30</td>
<td style="text-align: right;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">M1</td>
<td style="text-align: right;">0.04</td>
<td style="text-align: right;">-0.03</td>
<td style="text-align: right;">78.14</td>
<td style="text-align: right;">2506.28</td>
<td style="text-align: right;">2248.73</td>
<td style="text-align: right;">2266.13</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">All</td>
<td style="text-align: left;">M2</td>
<td style="text-align: right;">0.93</td>
<td style="text-align: right;">0.92</td>
<td style="text-align: right;">6.31</td>
<td style="text-align: right;">419.39</td>
<td style="text-align: right;">161.85</td>
<td style="text-align: right;">176.79</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">M3</td>
<td style="text-align: right;">0.98</td>
<td style="text-align: right;">0.97</td>
<td style="text-align: right;">2.12</td>
<td style="text-align: right;">302.43</td>
<td style="text-align: right;">44.88</td>
<td style="text-align: right;">57.17</td>
<td style="text-align: right;">27</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">All</td>
<td style="text-align: left;">M4</td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">0.36</td>
<td style="text-align: right;">257.54</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">9.42</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">M5</td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">0.31</td>
<td style="text-align: right;">258.87</td>
<td style="text-align: right;">1.33</td>
<td style="text-align: right;">7.65</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">0.184</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-mde-reg-coef" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;6: Summary of linear regression for observed species richness (S<sub>obs</sub>) as response variable and predicted species richness (S<sub>null</sub>) as predictor variable. The predicted species richness (S<sub>null</sub>) is the mean richness of 10,000 replications of mid-domain effect as null model. The estimated regression coefficients are represented as mean ± SE and the significance levels <code>***</code>, <code>**</code> and <code>*</code> correspond to <em>p</em>-value of &lt;0.001, &lt;0.01 and &lt;0.05, respectively.</caption>
<colgroup>
<col style="width: 12%">
<col style="width: 25%">
<col style="width: 21%">
<col style="width: 9%">
<col style="width: 7%">
<col style="width: 9%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Site</th>
<th style="text-align: left;">Intercept</th>
<th style="text-align: left;">S<sub>null</sub></th>
<th style="text-align: right;">F</th>
<th style="text-align: left;">df</th>
<th style="text-align: right;">p</th>
<th style="text-align: right;">R<sup>2</sup><sub>adj</sub></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">65.39 ± 46.18</td>
<td style="text-align: left;">0.4 ± 0.18</td>
<td style="text-align: right;">5.00</td>
<td style="text-align: left;">1,8</td>
<td style="text-align: right;">0.056</td>
<td style="text-align: right;">0.31</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">105.21 ± 10.45***</td>
<td style="text-align: left;">0.48 ± 0.06***</td>
<td style="text-align: right;">58.96</td>
<td style="text-align: left;">1,10</td>
<td style="text-align: right;">&lt;0.001</td>
<td style="text-align: right;">0.84</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">298.94 ± 85.69**</td>
<td style="text-align: left;">0.07 ± 0.37</td>
<td style="text-align: right;">0.03</td>
<td style="text-align: left;">1,16</td>
<td style="text-align: right;">0.862</td>
<td style="text-align: right;">-0.06</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">4.45 ± 29.65</td>
<td style="text-align: left;">0.85 ± 0.06***</td>
<td style="text-align: right;">170.20</td>
<td style="text-align: left;">1,29</td>
<td style="text-align: right;">&lt;0.001</td>
<td style="text-align: right;">0.85</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-sres-coef" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;7: Summary of linear regression for residual species richness (S<sub>res</sub>) as response variable and elevation (Elev) as predictor variable. The residual species richness (S<sub>res</sub>) is calculated as the difference between observed species richness (S) and null species richness (S<sub>null</sub>). The null species richness (S<sub>null</sub>) is defined as the mean of predicted species richness from 10,000 replications of the mid-domain effect null model. The estimated regression coefficients are represented as mean ± SE and the significance levels <code>***</code>, <code>**</code> and <code>*</code> correspond to <em>p</em>-value of &lt;0.001, &lt;0.01 and &lt;0.05, respectively. The elevation was transformed to kilometre units before modelling.</caption>
<colgroup>
<col style="width: 7%">
<col style="width: 17%">
<col style="width: 18%">
<col style="width: 16%">
<col style="width: 14%">
<col style="width: 6%">
<col style="width: 4%">
<col style="width: 6%">
<col style="width: 8%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Site</th>
<th style="text-align: left;">Intercept</th>
<th style="text-align: left;">Elev</th>
<th style="text-align: left;">Elev<sup>2</sup></th>
<th style="text-align: left;">Elev<sup>3</sup></th>
<th style="text-align: right;">F</th>
<th style="text-align: left;">df</th>
<th style="text-align: right;">p</th>
<th style="text-align: right;">R<sup>2</sup><sub>adj</sub></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Morni</td>
<td style="text-align: left;">237.64 ± 48.68**</td>
<td style="text-align: left;">-1019.86 ± 123.22***</td>
<td style="text-align: left;">680.36 ± 71.69***</td>
<td style="text-align: left;">NA</td>
<td style="text-align: right;">73.45</td>
<td style="text-align: left;">2,7</td>
<td style="text-align: right;">&lt;0.001</td>
<td style="text-align: right;">0.94</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chail</td>
<td style="text-align: left;">613.6 ± 67.77***</td>
<td style="text-align: left;">-819.31 ± 90.43***</td>
<td style="text-align: left;">269.92 ± 29.03***</td>
<td style="text-align: left;">NA</td>
<td style="text-align: right;">45.16</td>
<td style="text-align: left;">2,9</td>
<td style="text-align: right;">&lt;0.001</td>
<td style="text-align: right;">0.89</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Churdhar</td>
<td style="text-align: left;">2706.88 ± 101.88***</td>
<td style="text-align: left;">-1883.75 ± 82.19***</td>
<td style="text-align: left;">323.67 ± 16.05***</td>
<td style="text-align: left;">NA</td>
<td style="text-align: right;">697.41</td>
<td style="text-align: left;">2,15</td>
<td style="text-align: right;">&lt;0.001</td>
<td style="text-align: right;">0.99</td>
</tr>
<tr class="even">
<td style="text-align: left;">All</td>
<td style="text-align: left;">-222.61 ± 31.29***</td>
<td style="text-align: left;">591.16 ± 63.59***</td>
<td style="text-align: left;">-430.44 ± 37.07***</td>
<td style="text-align: left;">82.03 ± 6.44***</td>
<td style="text-align: right;">102.09</td>
<td style="text-align: left;">3,27</td>
<td style="text-align: right;">&lt;0.001</td>
<td style="text-align: right;">0.91</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</section>
<section id="supplementary-figures" class="level1">
<h1>Supplementary figures</h1>
<div class="cell">
<div class="cell-output-display">
<div id="fig-richness-linear" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-richness-linear-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;8: Exploratory analysis of the relationship between elevation and plant species richness. The scatter plot illustrates the univariate distribution of plant species richness across different elevations, providing initial insights into the potential relationship between these variables. Pearson’s correlation coefficient (<em>r</em>) was calculated to explore the strength and direction of the association. The regression line was fitted using the general linear model and the shaded region represents the 95% confidence intervals.</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div id="fig-qqplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-qqplot-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;9: Quantile-quantile (Q-Q) plot demonstrating model diagnostics for elevational patterns of species richness in (a) Morni Hills, (b) Chail WLS, (c) Churdhar WLS, and (d) All Sites. The plot shows the overall deviations from the simulation-based expected distribution with added tests for correct distribution (KS test), dispersion and outliers. The outliers are defined as values outside the simulation envelope. The plot assesses the goodness-of-fit between observed and expected quantiles, aiding in the evaluation of model assumptions and performance.</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div id="fig-residual-plot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-residual-plot-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;10: Residual plots showing the deviations in model residuals against the model predictions for elevational patterns of species richness in (a) Morni Hills, (b) Chail WLS, (c) Churdhar WLS, and (d) All Sites. The deviation from the uniformity (in y-direction) was estimated by comparing the empirical 0.25, 0.5 and 0.75 quantiles in y-direction (red solid lines) with the theoretical 0.25, 0.5 and 0.75 quantiles (dashed black line). The simulation-based outliers are highlighted as red stars.</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>#&gt; 
#&gt;  DHARMa nonparametric dispersion test via sd of residuals fitted vs.
#&gt;  simulated
#&gt; 
#&gt; data:  simulationOutput
#&gt; dispersion = 0.33287, p-value = 0.004
#&gt; alternative hypothesis: two.sided</code></pre>
</div>
<div class="cell-output-display">
<div id="fig-dispersion-all" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-dispersion-all-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;11: Simulation-based (n = 1000) dispersion test of the best model for All Site (full elevational gradient). This test compares the variance of the observed raw residuals (red line) against the variance of simulated residuals (histogram). The variances are scaled to the mean simulated variance. A significant ratio &gt; 1 indicates overdispersion, a significant ratio &lt; 1 underdispersion.</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div id="fig-sres-linear" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-sres-linear-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;12: Exploratory analysis of the relationship between elevation and residual species richness. The residual species richness (S<sub>res</sub>) was calculated by subtracting the predicted species richness (generated by the mid-domain effect null model) from the observed species richness (S) for each 100-m elevational band. The scatter plot illustrates the univariate distribution of residual species richness across different elevations, providing initial insights into the potential relationship between these variables. Pearson’s correlation coefficient (<em>r</em>) was calculated to explore the strength and direction of the association. The regression line was fitted using the general linear model and the shaded region represents the 95% confidence intervals.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="list">
<div id="ref-Acharyabk2011" class="csl-entry" role="listitem">
Acharya, B. K., Chettri, B., &amp; Vijayan, L. (2011). Distribution pattern of trees along an elevation gradient of eastern himalaya, india. <em>Acta Oecologica</em>, <em>37</em>(4), 329–336. <a href="https://doi.org/10.1016/j.actao.2011.03.005">https://doi.org/10.1016/j.actao.2011.03.005</a>
</div>
<div id="ref-APG2016" class="csl-entry" role="listitem">
Angiosperm Phylogeny Group, Chase, M. W., Christenhusz, M. J. M., Fay, M. F., Byng, J. W., Judd, W. S., … Stevens, P. F. (2016). An update of the <span>Angiosperm Phylogeny Group</span> classification for the orders and families of flowering plants: <span>APG IV</span>. <em>Botanical Journal of the Linnean Society</em>, <em>181</em>(1), 1–20. <a href="https://doi.org/10.1111/boj.12385">https://doi.org/10.1111/boj.12385</a>
</div>
<div id="ref-Balkrishna2018b" class="csl-entry" role="listitem">
Balkrishna, A., Joshi, B., Srivastava, A., &amp; Shukla, B. (2018). Phyto-resources of <span>Morni Hills, Panchkula, Haryana</span>. <em>Journal of Non-Timber Forest Products</em>, <em>25</em>(2), 91–98. <a href="https://doi.org/10.54207/bsmps2000-2018-p430i5">https://doi.org/10.54207/bsmps2000-2018-p430i5</a>
</div>
<div id="ref-Balkrishna2018a" class="csl-entry" role="listitem">
Balkrishna, A., Srivastava, A., Shukla, B., Mishra, R., &amp; Joshi, B. (2018). Medicinal plants of <span>Morni Hills, Shivalik Range, Panchkula, Haryana</span>. <em>Journal of Non-Timber Forest Products</em>, <em>25</em>(1), 1–14. <a href="https://doi.org/10.54207/bsmps2000-2018-ir3j0n">https://doi.org/10.54207/bsmps2000-2018-ir3j0n</a>
</div>
<div id="ref-COL2023" class="csl-entry" role="listitem">
Bánki, O., Roskov, Y., Döring, M., Ower, G., Vandepitte, L., Hobern, D., … Legume Phylogeny Working Group (LPWG). (2023). <em>Catalogue of life checklist (annual checklist 2023)</em>. Catalogue of Life. <a href="https://doi.org/10.48580/dfsr">https://doi.org/10.48580/dfsr</a>
</div>
<div id="ref-Beck2018" class="csl-entry" role="listitem">
Beck, H. E., Zimmermann, N. E., McVicar, T. R., Vergopolan, N., Berg, A., &amp; Wood, E. F. (2018). Present and future <span>Köppen-Geiger</span> climate classification maps at 1-km resolution. <em>Scientific Data</em>, <em>5</em>, 180214. <a href="https://doi.org/10.1038/sdata.2018.214">https://doi.org/10.1038/sdata.2018.214</a>
</div>
<div id="ref-Bhardwaj2017" class="csl-entry" role="listitem">
Bhardwaj, A. (2017). <em>Study on dynamics of plant bioresources in <span>Chail</span> wildlife sanctuary of <span>Himachal Pradesh</span></em> (PhD thesis, Forest Research Institute (Deemed) University; p. 342). Forest Research Institute (Deemed) University, Dehradun. Retrieved from <a href="http://hdl.handle.net/10603/175719">http://hdl.handle.net/10603/175719</a>
</div>
<div id="ref-Bhardwaj2014" class="csl-entry" role="listitem">
Bhardwaj, A., Verma, R. K., Rana, J. C., Thakur, K., &amp; Verma, J. (2014). <span class="nocase">Orchid diversity at Chail Wild life Sanctuary, Himachal Pradesh, Northwest Himalaya</span>. <em>Journal of Orchid Society of India</em>, <em>28</em>, 67–74.
</div>
<div id="ref-Bhattarai2004" class="csl-entry" role="listitem">
Bhattarai, K. R., Vetaas, O. R., &amp; Grytnes, J. A. (2004). Fern species richness along a central <span>Himalayan</span> elevational gradient, <span>Nepal</span>. <em>Journal of Biogeography</em>, <em>31</em>(3), 389–400. <a href="https://doi.org/10.1046/j.0305-0270.2003.01013.x">https://doi.org/10.1046/j.0305-0270.2003.01013.x</a>
</div>
<div id="ref-Bisht2022" class="csl-entry" role="listitem">
Bisht, M., Sekar, K. C., Mukherjee, S., Thapliyal, N., Bahukhandi, A., Singh, D., … Dey, D. (2022). Influence of anthropogenic pressure on the plant species richness and diversity along the elevation gradients of indian himalayan high-altitude protected areas. <em>Frontiers in Ecology and Evolution</em>, <em>10</em>, 751989. <a href="https://doi.org/10.3389/fevo.2022.751989">https://doi.org/10.3389/fevo.2022.751989</a>
</div>
<div id="ref-R-rWCVP" class="csl-entry" role="listitem">
Brown, M. J. M., Walker, B. E., Black, N., Govaerts, R. H. A., Ondo, I., Turner, R., &amp; Lughadha, E. N. (2023). <span class="nocase">rWCVP</span>: A companion <span>R</span> package for the world checklist of vascular plants. <em>New Phytologist</em>. <a href="https://doi.org/10.1111/nph.18919">https://doi.org/10.1111/nph.18919</a>
</div>
<div id="ref-Bryant2008" class="csl-entry" role="listitem">
Bryant, J. A., Lamanna, C., Morlon, H., Kerkhoff, A. J., Enquist, B. J., &amp; Green, J. L. (2008). Microbes on mountainsides: Contrasting elevational patterns of bacterial and plant diversity. <em>Proceedings of the National Academy of Sciences</em>, <em>105</em>(Suppl. 1), 11505–11511. <a href="https://doi.org/10.1073/pnas.0801920105">https://doi.org/10.1073/pnas.0801920105</a>
</div>
<div id="ref-Burnham2002" class="csl-entry" role="listitem">
Burnham, K. P., &amp; Anderson, D. R. (Eds.). (2002). <em>Model selection and multimodel inference</em> (2nd ed., p. 488). New York: Springer. <a href="https://doi.org/10.1007/b97636">https://doi.org/10.1007/b97636</a>
</div>
<div id="ref-R-rgbif" class="csl-entry" role="listitem">
Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., &amp; Ram, K. (2023). <span class="nocase">rgbif</span>: Interface to the global biodiversity information facility API (Version 3.7.7). CRAN. Retrieved from <a href="https://cran.r-project.org/package=rgbif">https://cran.r-project.org/package=rgbif</a>
</div>
<div id="ref-Champion1968" class="csl-entry" role="listitem">
Champion, H. G., &amp; Seth, S. K. (1968). <em>A revised survey of the forest types of <span>India</span></em> (p. 404). Delhi: Government of India.
</div>
<div id="ref-Chawla2008" class="csl-entry" role="listitem">
Chawla, A., Rajkumar, S., Singh, K. N., Lal, B., Singh, R. D., &amp; Thukral, A. K. (2008). Plant species diversity along an altitudinal gradient of bhabha valley in western himalaya. <em>Journal of Mountain Science</em>, <em>5</em>(2), 157–177. <a href="https://doi.org/10.1007/s11629-008-0079-y">https://doi.org/10.1007/s11629-008-0079-y</a>
</div>
<div id="ref-Choudhary2007" class="csl-entry" role="listitem">
Choudhary, A. K., Punam, Sharma, P. K., &amp; Chandel, S. (2007). Study on the physiography and biodiversity of <span>Churdhar</span> wildlife sanctuary of <span>Himachal Himalayas, India</span>. <em>Tigerpaper</em>, <em>34</em>, 27–32.
</div>
<div id="ref-Choudhary2012" class="csl-entry" role="listitem">
Choudhary, R. K., &amp; Lee, J. (2012). A floristic reconnaissance of <span>Churdhar</span> wildlife sanctuary of <span>Himachal Pradesh, India</span>. <em>Manthan</em>, <em>13</em>, 2–12.
</div>
<div id="ref-Chowdhery1984" class="csl-entry" role="listitem">
Chowdhery, H. J., &amp; Wadhwa, B. M. (1984). <em><span class="nocase">Flora of Himachal Pradesh</span></em> (Vol. 1–3). Calcutta: Botanical Survey of India.
</div>
<div id="ref-Collett1902" class="csl-entry" role="listitem">
Collett, H. (1902). <em><span>Flora Simlensis</span></em>. Simla: Thacker, Spink <span>&amp;</span> Co. <a href="https://doi.org/10.5962/bhl.title.138567">https://doi.org/10.5962/bhl.title.138567</a>
</div>
<div id="ref-Colwell1994" class="csl-entry" role="listitem">
Colwell, R. K., &amp; Hurtt, G. C. (1994). Nonbiological gradients in species richness and a spurious rapoport effect. <em>American Naturalist</em>, <em>144</em>(4), 570–595. <a href="https://doi.org/10.1086/285695">https://doi.org/10.1086/285695</a>
</div>
<div id="ref-Colwell2000" class="csl-entry" role="listitem">
Colwell, R. K., &amp; Lees, D. C. (2000). The mid-domain effect: Geometric constraints on the geography of species richness. <em>Trends in Ecology <span>&amp;</span> Evolution</em>, <em>15</em>(2), 70–76. <a href="https://doi.org/10.1016/s0169-5347(99)01767-x">https://doi.org/10.1016/s0169-5347(99)01767-x</a>
</div>
<div id="ref-Colwell2004" class="csl-entry" role="listitem">
Colwell, R. K., Rahbek, C., &amp; Gotelli, N. J. (2004). The mid-domain effect and species richness patterns: What have we learned so far? <em>American Naturalist</em>, <em>163</em>(3), E1–E23. <a href="https://doi.org/10.1086/382056">https://doi.org/10.1086/382056</a>
</div>
<div id="ref-Dhiman2020" class="csl-entry" role="listitem">
Dhiman, H., Saharan, H., &amp; Jakhar, S. (2020). Floristic diversity assessment and vegetation analysis of the upper altitudinal ranges of <span>Morni Hills, Panchkula, Haryana, India</span>. <em>Asian Journal of Conservation Biology</em>, <em>9</em>(1), 134–142.
</div>
<div id="ref-Dhiman2021" class="csl-entry" role="listitem">
Dhiman, H., Saharan, H., &amp; Jakhar, S. (2021). Study of invasive plants in tropical dry deciduous forests – biological spectrum, phenology, and diversity. <em>Forestry Studies</em>, <em>74</em>(1), 58–71. <a href="https://doi.org/10.2478/fsmu-2021-0004">https://doi.org/10.2478/fsmu-2021-0004</a>
</div>
<div id="ref-Di_Musciano2021" class="csl-entry" role="listitem">
Di Musciano, M., Zannini, P., Ferrara, C., Spina, L., Nascimbene, J., Vetaas, O. R., … Chiarucci, A. (2021). Investigating elevational gradients of species richness in a mediterranean plant hotspot using a published flora. <em>Frontiers of Biogeography</em>, <em>13</em>(3), e50007. <a href="https://doi.org/10.21425/f5fbg50007">https://doi.org/10.21425/f5fbg50007</a>
</div>
<div id="ref-Djordjevic2022" class="csl-entry" role="listitem">
Djordjević, V., Tsiftsis, S., Kindlmann, P., &amp; Stevanović, V. (2022). Orchid diversity along an altitudinal gradient in the central balkans. <em>Frontiers in Ecology and Evolution</em>, <em>10</em>, 929266. <a href="https://doi.org/10.3389/fevo.2022.929266">https://doi.org/10.3389/fevo.2022.929266</a>
</div>
<div id="ref-Dunn1996" class="csl-entry" role="listitem">
Dunn, P. K., &amp; Smyth, G. K. (1996). Randomized quantile residuals. <em>Journal of Computational and Graphical Statistics</em>, <em>5</em>(3), 236–244. <a href="https://doi.org/10.2307/1390802">https://doi.org/10.2307/1390802</a>
</div>
<div id="ref-Dunn2007" class="csl-entry" role="listitem">
Dunn, R. R., McCain, C. M., &amp; Sanders, N. J. (2007). When does diversity fit null model predictions? Scale and range size mediate the mid-domain effect. <em>Global Ecology and Biogeography</em>, <em>16</em>(3), 305–312. <a href="https://doi.org/10.1111/j.1466-8238.2006.00284.x">https://doi.org/10.1111/j.1466-8238.2006.00284.x</a>
</div>
<div id="ref-Duthie1903" class="csl-entry" role="listitem">
Duthie, J. F. (1903–1915). <em>Flora of the upper <span>Gangetic</span> plain, and of the adjacent <span>Siwalik</span> and <span class="nocase">sub-Himalayan</span> tracts</em> (Vol. 1–3). Calcutta: Superintendent of Government Printing. <a href="https://doi.org/10.5962/bhl.title.10981">https://doi.org/10.5962/bhl.title.10981</a>
</div>
<div id="ref-eFI2023" class="csl-entry" role="listitem">
eFI. (2023). <em>eFlora of <span>India</span>: Database of plants of <span>Indian</span> subcontinent</em>. Retrieved from <a href="https://efloraofindia.com/">https://efloraofindia.com/</a>
</div>
<div id="ref-Fick2017" class="csl-entry" role="listitem">
Fick, S. E., &amp; Hijmans, R. J. (2017). <span>WorldClim 2</span>: New 1-km spatial resolution climate surfaces for global land areas. <em>International Journal of Climatology</em>, <em>37</em>(12), 4302–4315. <a href="https://doi.org/10.1002/joc.5086">https://doi.org/10.1002/joc.5086</a>
</div>
<div id="ref-FOI2023" class="csl-entry" role="listitem">
FOI. (2023). <em>Flowers of <span>India</span></em>. Retrieved from <a href="http://www.flowersofindia.net/">http://www.flowersofindia.net/</a>
</div>
<div id="ref-Games1976" class="csl-entry" role="listitem">
Games, P. A., &amp; Howell, J. F. (1976). Pairwise multiple comparison procedures with unequal <span class="nocase">N’s</span> and/or variances: A monte carlo study. <em>Journal of Educational Statistics</em>, <em>1</em>(2), 113–125. <a href="https://doi.org/10.2307/1164979">https://doi.org/10.2307/1164979</a>
</div>
<div id="ref-Gaston2000" class="csl-entry" role="listitem">
Gaston, K. J. (2000). Global patterns in biodiversity. <em>Nature</em>, <em>405</em>(6783), 220–227. <a href="https://doi.org/10.1038/35012228">https://doi.org/10.1038/35012228</a>
</div>
<div id="ref-Gotelli2009" class="csl-entry" role="listitem">
Gotelli, N. J., Anderson, M. J., Arita, H. T., Chao, A., Colwell, R. K., Connolly, S. R., … Willig, M. R. (2009). Patterns and causes of species richness: A general simulation model for macroecology. <em>Ecology Letters</em>, <em>12</em>(9), 873–886. <a href="https://doi.org/10.1111/j.1461-0248.2009.01353.x">https://doi.org/10.1111/j.1461-0248.2009.01353.x</a>
</div>
<div id="ref-Govaerts2021" class="csl-entry" role="listitem">
Govaerts, R., Lughadha, E. N., Black, N., Turner, R., &amp; Paton, A. (2021). The <span class="nocase">World Checklist of Vascular Plants</span>, a continuously updated resource for exploring global plant diversity. <em>Scientific Data</em>, <em>8</em>(1), 215. <a href="https://doi.org/10.1038/s41597-021-00997-6">https://doi.org/10.1038/s41597-021-00997-6</a>
</div>
<div id="ref-Grau2007" class="csl-entry" role="listitem">
Grau, O., Grytnes, J. A., &amp; Birks, H. J. B. (2007). A comparison of altitudinal species richness patterns of bryophytes with other plant groups in nepal, central himalaya. <em>Journal of Biogeography</em>, <em>34</em>(11), 1907–1915. <a href="https://doi.org/10.1111/j.1365-2699.2007.01745.x">https://doi.org/10.1111/j.1365-2699.2007.01745.x</a>
</div>
<div id="ref-Grytnes2002" class="csl-entry" role="listitem">
Grytnes, J. A., &amp; Vetaas, O. R. (2002). Species richness and altitude: A comparison between null models and interpolated plant species richness along the <span>Himalayan</span> altitudinal gradient, <span>Nepal</span>. <em>American Naturalist</em>, <em>159</em>(3), 294–304. <a href="https://doi.org/10.1086/338542">https://doi.org/10.1086/338542</a>
</div>
<div id="ref-Guo2013" class="csl-entry" role="listitem">
Guo, Q., Kelt, D. A., Sun, Z., Liu, H., Hu, L., Ren, H., &amp; Wen, J. (2013). Global variation in elevational diversity patterns. <em>Scientific Reports</em>, <em>3</em>, 3007. <a href="https://doi.org/10.1038/srep03007">https://doi.org/10.1038/srep03007</a>
</div>
<div id="ref-Gupta1998" class="csl-entry" role="listitem">
Gupta, H. (1998). <em>Comparative studies on the medicinal and aromatic flora of <span>Churdhar</span> and <span>Rohtang</span> areas of <span>Himachal Pradesh</span></em> (Master’s thesis, Dr Yashwant Singh Parmar University of Horticulture <span>and</span> Forestry; p. 228). Dr Yashwant Singh Parmar University of Horticulture <span>and</span> Forestry, Solan. Retrieved from <a href="http://krishikosh.egranth.ac.in/handle/1/5810135063">http://krishikosh.egranth.ac.in/handle/1/5810135063</a>
</div>
<div id="ref-R-DHARMa" class="csl-entry" role="listitem">
Hartig, F. (2022). <span>DHARMa</span>: Residual diagnostics for hierarchical (multi-level / mixed) regression models (Version 0.4.6). CRAN. Retrieved from <a href="https://cran.r-project.org/package=DHARMa">https://cran.r-project.org/package=DHARMa</a>
</div>
<div id="ref-R-terra" class="csl-entry" role="listitem">
Hijmans, R. J. (2023). <span class="nocase">terra</span>: Spatial data analysis (Version 1.7.29). CRAN. Retrieved from <a href="https://cran.r-project.org/package=terra">https://cran.r-project.org/package=terra</a>
</div>
<div id="ref-Hilbe2014" class="csl-entry" role="listitem">
Hilbe, J. M. (2014). <em>Modeling count data</em>. Cambridge: Cambridge University Press. <a href="https://doi.org/10.1017/cbo9781139236065">https://doi.org/10.1017/cbo9781139236065</a>
</div>
<div id="ref-Homeier2010" class="csl-entry" role="listitem">
Homeier, J., Breckle, S.-W., Günter, S., Rollenbeck, R. T., &amp; Leuschner, C. (2010). Tree diversity, forest structure and productivity along altitudinal and topographical gradients in a species-rich ecuadorian montane rain forest. <em>Biotropica</em>, <em>42</em>(2), 140–148. <a href="https://doi.org/10.1111/j.1744-7429.2009.00547.x">https://doi.org/10.1111/j.1744-7429.2009.00547.x</a>
</div>
<div id="ref-Hu2016" class="csl-entry" role="listitem">
Hu, W., Wu, F., Gao, J., Yan, D., Liu, L., &amp; Yang, X. (2016). Influences of interpolation of species ranges on elevational species richness gradients. <em>Ecography</em>, <em>40</em>(10), 1231–1241. <a href="https://doi.org/10.1111/ecog.02534">https://doi.org/10.1111/ecog.02534</a>
</div>
<div id="ref-Humboldt1805" class="csl-entry" role="listitem">
Humboldt, A. von, &amp; Bonpland, A. (1805). <em>Essai sur la géographie des plantes</em> (p. 274). Paris: Chez Levrault, Schoell et Compagnie. <a href="https://doi.org/10.5962/bhl.title.9309">https://doi.org/10.5962/bhl.title.9309</a>
</div>
<div id="ref-Hurvich1989" class="csl-entry" role="listitem">
Hurvich, C. M., &amp; Tsai, C.-L. (1989). Regression and time series model selection in small samples. <em>Biometrika</em>, <em>76</em>(2), 297–307. <a href="https://doi.org/10.1093/biomet/76.2.297">https://doi.org/10.1093/biomet/76.2.297</a>
</div>
<div id="ref-IUCN2023" class="csl-entry" role="listitem">
IUCN. (2023). <em>The IUCN red list of threatened species</em>. Retrieved from <a href="https://www.iucnredlist.org">https://www.iucnredlist.org</a>
</div>
<div id="ref-Johnson2004" class="csl-entry" role="listitem">
Johnson, J. B., &amp; Omland, K. S. (2004). Model selection in ecology and evolution. <em>Trends in Ecology <span>&amp;</span> Evolution</em>, <em>19</em>(2), 101–108. <a href="https://doi.org/10.1016/j.tree.2003.10.013">https://doi.org/10.1016/j.tree.2003.10.013</a>
</div>
<div id="ref-R-rstatix" class="csl-entry" role="listitem">
Kassambara, A. (2023). <span class="nocase">rstatix</span>: Pipe-friendly framework for basic statistical tests (Version 0.7.2). CRAN. Retrieved from <a href="https://cran.r-project.org/package=rstatix">https://cran.r-project.org/package=rstatix</a>
</div>
<div id="ref-Kessler2011" class="csl-entry" role="listitem">
Kessler, M., Kluge, J., Hemp, A., &amp; Ohlemüller, R. (2011). A global comparative analysis of elevational species richness patterns of ferns. <em>Global Ecology and Biogeography</em>, <em>20</em>(6), 868–880. <a href="https://doi.org/10.1111/j.1466-8238.2011.00653.x">https://doi.org/10.1111/j.1466-8238.2011.00653.x</a>
</div>
<div id="ref-Khuroo2011" class="csl-entry" role="listitem">
Khuroo, A. A., Weber, E., Malik, A. H., Reshi, Z. A., &amp; Dar, G. H. (2011). Altitudinal distribution patterns of the native and alien woody flora in kashmir himalaya, india. <em>Environmental Research</em>, <em>111</em>(7), 967–977. <a href="https://doi.org/10.1016/j.envres.2011.05.006">https://doi.org/10.1016/j.envres.2011.05.006</a>
</div>
<div id="ref-Kumar2013" class="csl-entry" role="listitem">
Kumar, R. (2013). <em>Studies on plant biodiversity of <span>Chail</span> wildlife sanctuary in <span>Himachal Pradesh</span></em> (Master’s thesis, Dr Yashwant Singh Parmar University of Horticulture <span>and</span> Forestry; p. 119). Dr Yashwant Singh Parmar University of Horticulture <span>and</span> Forestry, Solan. Retrieved from <a href="http://krishikosh.egranth.ac.in/handle/1/91126">http://krishikosh.egranth.ac.in/handle/1/91126</a>
</div>
<div id="ref-Li2022" class="csl-entry" role="listitem">
Li, L., Xu, X., Qian, H., Huang, X., Liu, P., Landis, J. B., … Deng, T. (2022). Elevational patterns of phylogenetic structure of angiosperms in a biodiversity hotspot in eastern <span>Himalaya</span>. <em>Diversity and Distributions</em>, <em>28</em>(12), 2534–2548. <a href="https://doi.org/10.1111/ddi.13513">https://doi.org/10.1111/ddi.13513</a>
</div>
<div id="ref-Lomolino2001" class="csl-entry" role="listitem">
Lomolino, M. V. (2001). <span class="nocase">Elevation gradients of species-density: Historical and prospective views</span>. <em>Global Ecology and Biogeography</em>, <em>10</em>(1), 3–13. <a href="https://doi.org/10.1046/j.1466-822x.2001.00229.x">https://doi.org/10.1046/j.1466-822x.2001.00229.x</a>
</div>
<div id="ref-Manish2021" class="csl-entry" role="listitem">
Manish, K. (2021). Species richness, phylogenetic diversity and phylogenetic structure patterns of exotic and native plants along an elevational gradient in the himalaya. <em>Ecological Processes</em>, <em>10</em>, 64. <a href="https://doi.org/10.1186/s13717-021-00335-z">https://doi.org/10.1186/s13717-021-00335-z</a>
</div>
<div id="ref-Manish2017" class="csl-entry" role="listitem">
Manish, K., Pandit, M. K., Telwala, Y., Nautiyal, D. C., Koh, L. P., &amp; Tiwari, S. (2017). Elevational plant species richness patterns and their drivers across non-endemics, endemics and growth forms in the eastern <span>Himalaya</span>. <em>Journal of Plant Research</em>, <em>130</em>(5), 829–844. <a href="https://doi.org/10.1007/s10265-017-0946-0">https://doi.org/10.1007/s10265-017-0946-0</a>
</div>
<div id="ref-McCain2009" class="csl-entry" role="listitem">
McCain, C. M. (2009). Global analysis of bird elevational diversity. <em>Global Ecology and Biogeography</em>, <em>18</em>(3), 346–360. <a href="https://doi.org/10.1111/j.1466-8238.2008.00443.x">https://doi.org/10.1111/j.1466-8238.2008.00443.x</a>
</div>
<div id="ref-McCain2010" class="csl-entry" role="listitem">
McCain, C. M., &amp; Grytnes, J. A. (2010). Elevational gradients in species richness. In <em>Encyclopedia of life sciences (eLS)</em>. Chichester: Wiley. <a href="https://doi.org/10.1002/9780470015902.a0022548">https://doi.org/10.1002/9780470015902.a0022548</a>
</div>
<div id="ref-Oommen2005" class="csl-entry" role="listitem">
Oommen, M. A., &amp; Shanker, K. (2005). Elevational species richness patterns emerge from multiple local mechanisms in himalayan woody plants. <em>Ecology</em>, <em>86</em>(11), 3039–3047. <a href="https://doi.org/10.1890/04-1837">https://doi.org/10.1890/04-1837</a>
</div>
<div id="ref-Peters2016" class="csl-entry" role="listitem">
Peters, M. K., Hemp, A., Appelhans, T., Behler, C., Classen, A., Detsch, F., … Steffan-Dewenter, I. (2016). Predictors of elevational biodiversity gradients change from single taxa to the multi-taxa community level. <em>Nature Communications</em>, <em>7</em>, 13736. <a href="https://doi.org/10.1038/ncomms13736">https://doi.org/10.1038/ncomms13736</a>
</div>
<div id="ref-POWO2022" class="csl-entry" role="listitem">
POWO. (2022). <em>Plants of the world online</em>. Kew: Royal Botanic Gardens. Retrieved from <a href="http://www.plantsoftheworldonline.org/">http://www.plantsoftheworldonline.org/</a>
</div>
<div id="ref-Qian2022" class="csl-entry" role="listitem">
Qian, H., Kessler, M., &amp; Vetaas, O. R. (2022). Pteridophyte species richness in the central <span>Himalaya</span> is limited by cold climate extremes at high elevations and rainfall seasonality at low elevations. <em>Ecology and Evolution</em>, <em>12</em>(5), e8958. <a href="https://doi.org/10.1002/ece3.8958">https://doi.org/10.1002/ece3.8958</a>
</div>
<div id="ref-R-base" class="csl-entry" role="listitem">
R Core Team. (2023). R: A language and environment for statistical computing (Version 4.3.0). Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="https://www.R-project.org/">https://www.R-project.org/</a>
</div>
<div id="ref-Radha2019" class="csl-entry" role="listitem">
Radha, Puri, S., Chandel, K., Pundir, A., Thakur, M. S., Chauhan, B., … Kumar, S. (2019). Diversity of ethnomedicinal plants in <span>Churdhar</span> wildlife sanctuary of district <span class="nocase">Sirmour of Himachal Pradesh, India</span>. <em>Journal of Applied Pharmaceutical Science</em>, <em>9</em>(11), 48–53. <a href="https://doi.org/10.7324/japs.2019.91106">https://doi.org/10.7324/japs.2019.91106</a>
</div>
<div id="ref-Rahbek1995" class="csl-entry" role="listitem">
Rahbek, C. (1995). The elevational gradient of species richness: A uniform pattern? <em>Ecography</em>, <em>18</em>(2), 200–205. <a href="https://doi.org/10.1111/j.1600-0587.1995.tb00341.x">https://doi.org/10.1111/j.1600-0587.1995.tb00341.x</a>
</div>
<div id="ref-Rahbek2005" class="csl-entry" role="listitem">
Rahbek, C. (2005). The role of spatial scale and the perception of large-scale species-richness patterns. <em>Ecology Letters</em>, <em>8</em>(2), 224–239. <a href="https://doi.org/10.1111/j.1461-0248.2004.00701.x">https://doi.org/10.1111/j.1461-0248.2004.00701.x</a>
</div>
<div id="ref-Rahbek2019a" class="csl-entry" role="listitem">
Rahbek, C., Borregaard, M. K., Colwell, R. K., Dalsgaard, B., Holt, B. G., Morueta-Holme, N., … Fjeldså, J. (2019). Humboldt’s enigma: What causes global patterns of mountain biodiversity? <em>Science</em>, <em>365</em>(6458), 1108–1113. <a href="https://doi.org/10.1126/science.aax0149">https://doi.org/10.1126/science.aax0149</a>
</div>
<div id="ref-Rahmani2016" class="csl-entry" role="listitem">
Rahmani, A. R., Islam, M. Z., &amp; Kasambe, R. M. (2016). <em>Important bird and biodiversity areas in <span>India</span>: Priority sites for conservation (revised and updated)</em> (2nd ed., Vol. 1, p. 1992). Mumbai, India <span>and</span> Cambridge, UK: Bombay Natural History Society <span>and</span> BirdLife International.
</div>
<div id="ref-Rana2019ecy" class="csl-entry" role="listitem">
Rana, S. K., Gross, K., &amp; Price, T. D. (2019). Drivers of elevational richness peaks, evaluated for trees in the east <span>Himalaya</span>. <em>Ecology</em>, <em>100</em>(1), e02548. <a href="https://doi.org/10.1002/ecy.2548">https://doi.org/10.1002/ecy.2548</a>
</div>
<div id="ref-Rana2019ecs" class="csl-entry" role="listitem">
Rana, S. K., Price, T. D., &amp; Qian, H. (2019). Plant species richness across the <span>Himalaya</span> driven by evolutionary history and current climate. <em>Ecosphere</em>, <em>10</em>(11), e02945. <a href="https://doi.org/10.1002/ecs2.2945">https://doi.org/10.1002/ecs2.2945</a>
</div>
<div id="ref-Rana2017" class="csl-entry" role="listitem">
Rana, S. K., &amp; Rawat, G. S. (2017). Database of <span>Himalayan</span> plants based on published floras during a century. <em>Data</em>, <em>2</em>(4), 36. <a href="https://doi.org/10.3390/data2040036">https://doi.org/10.3390/data2040036</a>
</div>
<div id="ref-Rana2019gbif" class="csl-entry" role="listitem">
Rana, S. K., &amp; Rawat, G. S. (2019). <em>Database of vascular plants of <span>Himalaya</span></em>. Dehradun: Wildlife Institute of India. <a href="https://doi.org/10.15468/zdeuix">https://doi.org/10.15468/zdeuix</a>
</div>
<div id="ref-Rodriguez-Quiel2022" class="csl-entry" role="listitem">
Rodrı́guez-Quiel, E. E., Kluge, J., Mendieta-Leiva, G., &amp; Bader, M. Y. (2022). Elevational patterns in tropical bryophyte diversity differ among substrates: A case study on baru volcano, panama. <em>Journal of Vegetation Science</em>, <em>33</em>(3), e13136. <a href="https://doi.org/10.1111/jvs.13136">https://doi.org/10.1111/jvs.13136</a>
</div>
<div id="ref-Ruxton2008" class="csl-entry" role="listitem">
Ruxton, G. D., &amp; Beauchamp, G. (2008). Time for some a priori thinking about post hoc testing. <em>Behavioral Ecology</em>, <em>19</em>(3), 690–693. <a href="https://doi.org/10.1093/beheco/arn020">https://doi.org/10.1093/beheco/arn020</a>
</div>
<div id="ref-Singh2014" class="csl-entry" role="listitem">
Singh, N., &amp; Vashistha, B. D. (2014). Flowering plant diversity and ethnobotany of <span>Morni Hills, Siwalik Range, Haryana, India</span>. <em>International Journal of Pharma and Bio Sciences</em>, <em>5</em>(2), B214–B222.
</div>
<div id="ref-Steinbauer2018" class="csl-entry" role="listitem">
Steinbauer, M. J., Grytnes, J.-A., Jurasinski, G., Kulonen, A., Lenoir, J., Pauli, H., … Wipf, S. (2018). Accelerated increase in plant species richness on mountain summits is linked to warming. <em>Nature</em>, <em>556</em>(7700), 231–234. <a href="https://doi.org/10.1038/s41586-018-0005-6">https://doi.org/10.1038/s41586-018-0005-6</a>
</div>
<div id="ref-Stevens1992" class="csl-entry" role="listitem">
Stevens, G. C. (1992). The elevational gradient in altitudinal range: An extension of <span class="nocase">Rapoport’s</span> latitudinal rule to altitude. <em>American Naturalist</em>, <em>140</em>(6), 893–911. <a href="https://doi.org/10.1086/285447">https://doi.org/10.1086/285447</a>
</div>
<div id="ref-Subramani2014" class="csl-entry" role="listitem">
Subramani, S. P., Kapoor, K. S., &amp; Goraya, G. S. (2014). Additions to the floral wealth of <span>Sirmaur</span> district, <span>Himachal Pradesh</span> from <span>Churdhar</span> wildlife sanctuary. <em>Journal of Threatened Taxa</em>, <em>6</em>(11), 6427–6452. <a href="https://doi.org/10.11609/jott.o2845.6427-52">https://doi.org/10.11609/jott.o2845.6427-52</a>
</div>
<div id="ref-Thakur2021a" class="csl-entry" role="listitem">
Thakur, U., Bisht, N. S., Kumar, M., &amp; Kumar, A. (2021). Influence of altitude on diversity and distribution pattern of trees in <span>Himalayan</span> temperate forests of <span>Churdhar</span> wildlife sanctuary, <span>India</span>. <em>Water, Air, <span>&amp;</span> Soil Pollution</em>, <em>232</em>, 205. <a href="https://doi.org/10.1007/s11270-021-05162-8">https://doi.org/10.1007/s11270-021-05162-8</a>
</div>
<div id="ref-Thorne2022" class="csl-entry" role="listitem">
Thorne, J. H., Choe, H., Dorji, L., Yangden, K., Wangdi, D., Phuntsho, Y., &amp; Beardsley, K. (2022). Species richness and turnover patterns for tropical and temperate plants on the elevation gradient of the eastern himalayan mountains. <em>Frontiers in Ecology and Evolution</em>, <em>10</em>, 942759. <a href="https://doi.org/10.3389/fevo.2022.942759">https://doi.org/10.3389/fevo.2022.942759</a>
</div>
<div id="ref-Trigas2013" class="csl-entry" role="listitem">
Trigas, P., Panitsa, M., &amp; Tsiftsis, S. (2013). Elevational gradient of vascular plant species richness and endemism in crete – the effect of post-isolation mountain uplift on a continental island system. <em><span>PLoS</span> <span>ONE</span></em>, <em>8</em>(3), e59425. <a href="https://doi.org/10.1371/journal.pone.0059425">https://doi.org/10.1371/journal.pone.0059425</a>
</div>
<div id="ref-Turland2018" class="csl-entry" role="listitem">
Turland, N., Wiersema, J., Barrie, F., Greuter, W., Hawksworth, D., Herendeen, P., … Smith, G. (Eds.). (2018). <em><span class="nocase">International Code of Nomenclature for algae, fungi, and plants (Shenzhen Code)</span></em> (p. 254). Glashütten, Germany: Koeltz Botanical Books. <a href="https://doi.org/10.12705/code.2018">https://doi.org/10.12705/code.2018</a>
</div>
<div id="ref-Vetaas2002" class="csl-entry" role="listitem">
Vetaas, O. R., &amp; Grytnes, J. A. (2002). Distribution of vascular plant species richness and endemic richness along the himalayan elevation gradient in nepal. <em>Global Ecology and Biogeography</em>, <em>11</em>(4), 291–301. <a href="https://doi.org/10.1046/j.1466-822x.2002.00297.x">https://doi.org/10.1046/j.1466-822x.2002.00297.x</a>
</div>
<div id="ref-Walter1967" class="csl-entry" role="listitem">
Walter, H., &amp; Lieth, H. H. F. (1967). <em>Klimadiagramm-weltatlas</em>. Jena: G. Fischer.
</div>
<div id="ref-Welch1951" class="csl-entry" role="listitem">
Welch, B. L. (1951). On the comparison of several mean values: An alternative approach. <em>Biometrika</em>, <em>38</em>(3/4), 330–336. <a href="https://doi.org/10.2307/2332579">https://doi.org/10.2307/2332579</a>
</div>
<div id="ref-R-tidyverse" class="csl-entry" role="listitem">
Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L., François, R., … Yutani, H. (2019). Welcome to the tidyverse. <em>Journal of Open Source Software</em>, <em>4</em>(43), 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>